---
title: Elucidating the Transcriptome of Turkey Hemorrhagic Enteritis Virus
output:
  word_document: default
  pdf_document: default
geometry: margin=1in
font-size: 11pt
header-includes:
  - \usepackage{helvet}
  - \renewcommand*\familydefault{\sfdefault}
  - \usepackage{setspace}
  - \doublespacing
  - \usepackage[left]{lineno}
  - \linenumbers
  - \usepackage{booktabs}
  - \usepackage{longtable}
  - \usepackage{array}
  - \usepackage{multirow}
  - \usepackage{wrapfig}
  - \usepackage{float}
  - \usepackage{colortbl}
  - \usepackage{pdflscape}
  - \usepackage{tabu}
  - \usepackage{threeparttable}
  - \usepackage{threeparttablex}
  - \usepackage[normalem]{ulem}
  - \usepackage{makecell}
  - \usepackage{xcolor}
bibliography: transcriptome_refs.bib
csl: asm.csl
editor_options:
  chunck_output_type: console
---
\vspace{5mm}

**Running Title:** Novel Insights into Turkey Hemorrhagic Enteritis Virus Transcriptome

\vspace{10mm}
Abraham Quaye^1\*^, Bret Pickett^\*^, Joel S. Griffitts^\*^, Bradford K. Berges^\*^, Brian D. Poole${^\dagger}$^\*^   

\vspace{5mm}

^\*^Department of Microbiology and Molecular Biology, Brigham Young University       
^1^First-author      
${^\dagger}$ Corresponding Author   

\vspace{5mm}
**Corresponding Author Information**     
[brian_poole@byu.edu](mailto:brian_poole@byu.edu)       
Department of Microbiology and Molecular Biology,       
4007 Life Sciences Building (LSB),    
Brigham Young University,   
Provo, Utah  
\newpage

```{r call_libs, echo=FALSE,include=FALSE, message=FALSE, warning=FALSE}
library(knitr)
library(tidyverse)
library(tibble)
library(scales)
library(flextable)
library(english)
opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
knitr::opts_chunk$set(tab.cap.pre = "", tab.cap.sep = "")

inline_hook <- function(x) {
  if(is.numeric(x)){
    formatted <- format(x, big.mark = ",")
  }else{formatted <- x}
  paste0("**", formatted, "**")
}
knit_hooks$set(inline = inline_hook)
```

```{r inline_values, echo=FALSE,include=FALSE, message=FALSE, warning=FALSE}

# extract paired-end sequencing read length
bam_info <- read_tsv("results/sgseq/bam_extracted_sampleData.tsv",
                     show_col_types = F) 
seq_len <- bam_info %>% select(read_length) %>% slice_head() %>% pull()


## load in variables to be used inline 
source("scripts/r/bam_file_analysis.R")
source("scripts/r/thev_splicing_fullMap.R")
source("scripts/r/abundance_analyses.R")
source("scripts/r/reg_by_reg_plots.R")


cov_thev <- all_covs %>% filter(organism == "thev")

cov_4hr <- cov_thev %>% filter(timepoint == "4hpi") %>%
              select(total_mapped) %>% pull() %>% as.numeric(.)
cov_72hr <- cov_thev %>% filter(timepoint == "72hpi") %>%
              select(total_mapped) %>% pull() %>% as.numeric(.)
cov_72hr <- round(cov_72hr / 1e6, 1)

total_mapped <- round(sum(as.numeric(cov_thev$total_mapped)) / 1e6, 1)

avg_tot_count <- round(mean(total_reads$total_reads) / 1e6, 1)


### inline code for figure 2: depth graph
dep_4 <- as.numeric(tab1$"4h.p.i"[4])
dep_72 <- as.numeric(str_replace(tab1$"72h.p.i"[4], ",", "")) %>% comma(.)

## inline code of junction stats
tot_jun_4 <- tab1$`4h.p.i`[5]
tot_jun_72 <- tab1$`72h.p.i`[5]
mean_juncreads_4 <- tab1$`4h.p.i`[7]
mean_juncreads_72 <- round(as.numeric(tab1$`72h.p.i`[7]), 1)
max_junread72 <- bulk_junc_stats %>% filter(timepoint == "72hpi") %>%
  pull(read_count) %>% max(.) %>% comma(.)

most_sig_12 <- bulk_junc_stats %>% filter(timepoint == "12hpi", region == "E2") %>% pull(read_count) %>% sum()/as.numeric(tab1$`12h.p.i`[6])*100

most_sig_24 <- bulk_junc_stats %>% filter(timepoint == "24hpi", region == "MLP") %>% pull(read_count) %>% sum()/as.numeric(tab1$`24h.p.i`[6])*100

most_sig_72 <- bulk_junc_stats %>% filter(timepoint == "72hpi", region == "MLP") %>% pull(read_count) %>% sum()/as.numeric(tab1$`72h.p.i`[6])*100
```
## ABSTRACT        
**Background:** Hemorrhagic enteritis (HE) is a disease affecting 6-12-week-old turkeys characterized by _immunosuppression (IS)_ and bloody diarrhea. This disease is caused by _Turkey Hemorrhagic Enteritis Virus (THEV)_ of which avirulent strains (THEV-A) that do not cause HE but retain the immunosuppressive ability have been isolated. The THEV-A Virginia Avirulent Strain (VAS) is still used as a live vaccine despite its immunosuppressive properties. _Our objective is to understand the genetic basis by which VAS induces IS_. The _transcriptome of THEV_ was studied to set the stage for further experimentation with specific viral genes that may mediate IS.      
**Methods:** After infecting a turkey B-cell line (MDTC-RP19) with the VAS vaccine strain, samples in triplicates were collected at 4-, 12-, 24-, and 72-hours post-infection. Total RNA was subsequently extracted, and poly-A-tailed mRNA sequencing done. After trimming the raw sequencing reads with the `FastQC`, reads were mapped to the THEV genome using `Hisat2` and transcripts assembled with `StringTie`. An in-house script was used to consolidate transcripts from all time-points, generating the final transcriptome. PCR, gel electrophoresis, and Sanger sequencing were used to validate all identified splice junctions.      
**Results and Conclusions:** A total of `r total_mapped` million reads mapped to THEV genome providing good coverage/depth, leaving no regions unmapped. All predicted genes in the genome were represented. In keeping with all adenoviruses, all transcripts were spliced with either with 5'- or 3'-multi exon UTRs hitherto unknown. *~~Thirteen~~* novel exons were identified which were validated by PCR and Sanger sequencing. The splicing patterns strongly suggest that there are *~~three~~* main promoters (E1, E3, and major late promoters) driving expression of most of the genes with *~~two~~* possible minor promoters driving single genes (ORF7 and ORF8). This RNA-sequencing experiment is the first study of THEV gene expression to date. In keeping with other Adenoviruses, almost all THEV genes are spliced, and several genes are expressed as one transcription unit under a single promoter. This insight into THEV's transcriptome may allow the engineering of the VAS to provide immune protection with less or no associated IS. 
\newpage

## INTRODUCTION   
Adenoviruses (AdVs) are non-enveloped icosahedral-shaped DNA viruses, causing infection in virtually all vertebrates. Their double-stranded linear DNA genomes range between 26 and 45kb in size, producing a broad repertoire of transcripts via a highly complex alternative splicing pattern [@Davison2003; @Harrach2008]. The AdV genome is one of the most optimally economized; both the forward and reverse DNA strands harbor protein-coding genes, making it highly gene-dense. There are 16 genes termed "genus-common" that are homologous in all AdVs; these are thought to be inherited from a common ancestor. All other genes are termed "genus-specific". "Genus-specific" genes tend to be located at the termini of the genome while "genus-common" genes are usually central [@Davison2003]. This pattern is observed in *Adenoviridae*, *Poxviridae*, and *Herpesviridae* [@Davison2003; @Upton2003; @McGeoch1999]. The family *Adenoviridae* consists of five genera: *Mastadenovirus* (MAdV), *Aviadenovirus*, *Atadenovirus*, *Ichtadenovirus*, and *Siadenovirus* (SiAdV) [@Harrach2011; @Guimet2016]. Currently, there are three recognized members of the genus SiAdV: frog adenovirus 1, raptor adenovirus 1, and turkey adenovirus 3 also called turkey hemorrhagic enteritis virus (THEV) [@Harrach2011; @Kovács2011; @Davison2000; @Kovács2010; @Katoh2009]. Members of SiAdV have the smallest genome size (~26 kb) and gene content (~23 genes) of all known AdVs, and many “genus-specific” putative genes of unknown functions have been annotated (see __Figure 1__)[@Harrach2008; @Kovács2011; @Davison2003].

Virulent strains (THEV-V) and avirulent strains (THEV-A) of THEV are serologically indistinguishable, infecting turkeys, chickens, and pheasants and the THEV-V cause different clinical diseases in these birds [@Harrach2008; @Beach2006]. In turkeys, the THEV-V cause hemorrhagic enteritis (HE), a debilitating acute disease affecting predominantly 6-12-week-old turkeys characterized by immunosuppression (IS), weight loss, intestinal lesions leading to bloody diarrhea, splenomegaly, and up to 80% mortality [@Beach2006; @Beach2009a; @Gross1967]. HE is the most economically significant disease caused by any strain of THEV [@Beach2006]. While the current vaccine strain (a THEV-A isolated from a pheasant, Virginia Avirulent Strain [VAS]) have proven effective at preventing HE in young turkey poults, it still retains the immunosuppressive ability. Thus, vaccinated birds are rendered more susceptible to opportunistic infections and death than unvaccinated cohorts leading to substantial economic losses [@Beach2006; @Rautenschlein2000; @Larsen1985; @Dhama2017]. The induced IS also interferes with vaccination schemes for other infections of turkeys [@Beach2006; @Rautenschlein2000]. To eliminate this immunossupressive side-effect of the vaccine, a thorough investigation of the culprit viral factors (genes) mediating this phenomenon is essential. However, the transcriptome (splicing and gene expression patterns) of THEV has not been characterized, making the investigation of specific viral genes for possible roles in causing IS impractical. A well-characterized transcriptome of THEV is required to enable the next leap forward in THEV research - experimentation with specific viral genes that may mediate IS.

Myriads of studies have elucidated the AdV transcriptome in fine detail [@Donovan2020; @Zhao2014]. However, a large preponderance of studies focus on MAdVs - specifically human AdVs - thus, most of the current knowledge regarding AdV gene expression and replication is based on MAdV studies, which is generalized for all other AdVs [@Guimet2016; @Wolfrum2013]. MAdV genes are transcribed in a temporal manner; therefore, genes are categorized into five early transcription units (E1A, E1B, E2, E3, and E4), two intermediate (IM) units (pIX and IVa2), and one major late unit (MLTU), which generates five families of late mRNAs (L1-L5). An additional gene (UXP or U exon) is located on the reverse strand. The early genes encode non-structural proteins such as enzymes or host cell modulating proteins, primarily involved in DNA replication or providing the necessary intracellular niche for optimal replication while late genes encode structural proteins. The immediate early gene E1A is expressed first, followed by the the delayed early genes, E1B, E2, E3 and E4. Then the intermediate early genes, IVa2 and pIX are expressed followed by the late genes [@Zhao2014; @Donovan2020; @Guimet2016]. MAdV makes an extensive use of alternative RNA splicing to produce a very complex array of mRNAs; all but pIX mRNA undergo at least one splicing event. The MLTU produces over 20 distinct splice variants all of which contain three non-coding exons at the 5'-end (collectively known as the tripartite leader, TPL) [@Donovan2020; @Zhao2014]. There is also an alternate 5' three non-coding exons present in varying amounts on a subset of MLTU mRNAs (known as the x-, y- and z-leaders). Lastly, there is the i-leader exon, which is infrequently included between the second and third TPL exons, and codes for the i-leader protein [@Falvey1983]. Thus, the MLTU produces a complex repertoire of mRNA with diverse 5'-UTRs spliced onto different 3' coding exons which are grouped into five different 3'-end classes (L1-L5). Each transcription unit (TU) contains its own promoter driving the expression of all the array of mRNA transcripts produced via alternative splicing of the genes encoded in the unit[@Zhao2014; @Donovan2020; @Guimet2016]. Almost all AdV mRNAs are generated by the excision of one or more introns and most of these introns are located in the 5' or 3' UTRs of pre-mRNA. Thus the viral introns scarcely interrupt the open reading frames (ORFs) [@Davison2003; @Zhao2014]. 

High throughput sequencing methods have facilitated the discovery of many novel transcribed regions and splicing isoforms. It is also a very powerful tool to study alternative splicing under different conditions at an unparalleled depth [@Djebali2012; @Zhao2014]. In this paper, a paired-end deep sequencing experiment was performed to characterize for the first time, the transcriptome of THEV (VAS vaccine strain) during different phases of the infection, yielding the first THEV splicing map. Our paired-end sequencing allowed for reading `r seq_len` bp long high quality (mean Phred Score of 36) sequences from each end of cDNA fragments, which were mapped to the genome of THEV. The generated data from our paired-end sequencing experiment should thus be reliable.
\newpage

## RESULTS   
**Overview of sequencing data and analysis pipeline outputs**     
A previous study by Zeinab *et al* showed that almost all THEV transcripts were detectable beginning at 4 hours [@Aboezz2019]. Therefore, infected MDTC-RP19 cells were harvested at 4-, 12-, 24-, and 72-hours post-infection(h.p.i) to ensure an amply wide time window to sample all transcripts. Our paired-end RNA sequencing (RNA-seq) experiment yielded an average of `r avg_tot_count` million total reads of `r seq_len`bp in length per time-point, which were simultaneously mapped to both the virus (THEV) and host (*M.gallopavo*) genomes using the `Hisat2` [@Pertea2016] alignment program. A total of `r total_mapped` million reads from all time-points mapped to the virus genome; this provided good coverage/depth, leaving no regions unmapped. The mapped reads to the virus genome increased substantially from `r cov_4hr` reads at 4 h.p.i to `r cov_72hr` million reads at 72 h.p.i (__Table 1__, __Figure 2a__). From the mapped reads, we identified an overall total of `r total_unq_jncs_submit %>% nrow() %>% comma(.)` unique THEV splice junctions from all time-points, with splice junctions from the later time-points being supported by significantly more sequence reads than earlier time-points. For example all the `r tot_jun_4` unique junctions at 4 h.p.i had less than 10 reads supporting each one, averaging a mere `r mean_juncreads_4` reads/junction. Conversely, the `r tot_jun_72` unique junctions at 72 h.p.i averaged `r mean_juncreads_72` reads/junction, some junctions having coverage as high as `r max_junread72` reads. The substantial increases in splice junctions and mapping reads to the THEV genome over time denotes an active infection, and correlates with our qPCR assay quantifying the total number of viral genome copies over time (__Figure 2b__). Using `StringTie` [@Pertea2016], an assembler of RNA-seq alignments into potential transcripts, the mapped reads for each time-point were assembled into transcripts using the genomic location of the predicted THEV ORFs as a guide. In the final consolidated transcriptome of THEV, a composite of all unredundant transcripts from all time points, we counted a total of `r nrow(spliced_gtf)` transcripts all of which are novel. Although some exons in some transcripts match the predicted ORFs exactly, most of our identified exons are longer, spanning multiple predicted ORFs (__Figure 3__). The complete list of unique splice junctions mapped to THEV's genome has been submitted to the National Center for Biotechnology Information Gene Expression Omnibus (http://www.ncbi.nlm.nih.gov/geo) under **accession no. XXXXXX**.

**Changes in THEV splicing profile over time**   
AdV gene expression occurs under exquisite temporal control, supervised by designated promoters for each transcription unit (TU). Each promoter typically produce one or few pre-mRNAs that undergo alternative splicing to yield the manifold repertoire of complex transcripts characteristic of AdVs. To evaluate the activity of each promoter over time, Firstly, `Ballgown` - a program for statistical analysis of assembled transcriptomes, was used to estimate and normalize expression levels of all transcripts in Fragments Per Kilobase of transcript per Million mapped reads (FPKM) units over time. Very few unique splice junctions, reads, and transcripts were counted at 4 h.p.i; hence, this time point was excluded in this analysis.This analysis revealed DBP - from the E2 region - to be the most significantly expressed at 12 h.p.i, whereas MLP region transcripts were most abundantly expressed at 24 and 72 h.p.i as expected (__Figure 4a__). Also, our analysis of the FPKM values of transcripts by region showed a similar pattern: the E2 region as the most significantly overexpressed at 12 h.p.i, after which the MLP region assumes predominance (__Figure 4b__).  
Secondly, we estimated relative abundances of all splice junctions for each time point, counting as significantly expressed only those with coverage of at least 1% of the total splice junction reads counted at the given time point. At 12 h.p.i, `r nrow(sig_12_juncs)` junctions meet the 1% threshold, and were predominantly early region (E1, E2, and E3) junctions, albeit a significant proportion are MLP-derived (__Table 2a__). Junctions from the E2 region were the most abundant at this time point, constituting `r round(most_sig_12, 1)`% of all junctions. The two most abundant junctions at 12 h.p.i were maintained at 24 h.p.i also as the most significantly expressed. However, the MLP-derived junctions were unsurprisingly the most preponderant overall at this time point, accounting for `r round(most_sig_24, 1)`% of all the junction reads counted (__Table 2b__). At 72 h.p.i, the trend of increased activity of the MLP continued as expected; at this time, the MLP-derived junctions was not only the most abundant overall - accounting for `r round(most_sig_72, 1)`% of all junctions reads - but also contained the most significantly expressed individual junctions (__Table 2c__. Also see __Supplementary Tables 1a-c__; __Figure 4c__). When we limited this analysis to only junctions in the final transcriptome, the relative abundances of the junctions for each region over time was substantially similar to the pattern seen with all the junctions included (__Figure 4d__).  
Furthermore, we analyzed splice donor and acceptor site nucleotide usage over time to investigate any peculiarities that THEV may show, generally or over the course of the infection. This revealed that, with the exception of one junction (which was a T-G donor-acceptor pair), all splice donor-acceptor pairs in the final transcriptome were the canonical T-A pair.

**Early Region 1 (E1) transcripts**. This region in MAdVs is the first transcribed after successful entry of the viral DNA into the host cell nucleus, albeit at low levels [@Zhao2014]. The host transcription machinery solely mediates the transcription of this region. After their translation, the E1 proteins in concert with a myriad of host transcription factors activate the other viral promoters [@Guimet2016]. Only two ORFs (ORF1 [sialidase] and Hyd) are predicted in this region; however, we discovered `r as.character(english(nrow(filter(spliced_gtf, region == "E1"))))` novel transcripts in this region, which collectively contain `r nrow(reg_e1_brkdown)` unique splice junctions (__Figure 5__). Most of the encoded proteins of the novel transcripts are distinct from the predicted ORFs, although they all have the potential to encode the Hyd protein as the 3'-most coding sequence (CDS) if the first start codon (SSC) is skipped. The 5'-most CDS of TRXPT_1 is multi-exonic, producing a 17.9 kilodalton (kDa) protein of 160 residues [amino acids (aa)]. The CDS begins in the first exon, starting at position 211, spans the second exon, and terminates in the third exon at position 2312. From the 5'-most SSC, TRXPT_2 encodes the largest protein in this region ——  a 64.3 kDa, 580 aa protein with the same SSC as TRXPT_1 (position 211). This CDS spans almost the entire predicted ORF1 and Hyd, coming short in two regards: it is spliced from 1655 to 1964 (ORF1's C-terminus, including the stop codon), and it's stop codon (STC; position 2312) is 13 bp short of the Hyd STC. However, it has an SSC 102 bp upstream and in-frame with ORF1's predicted SSC. The N-terminus of TRXPT_2 CDS therefore, shares substantial protein sequence similarity with ORF1 but Hyd and TRXPT_2 are not in-frame; hence no protein sequence similarity. TRXPT_3 is almost identical to TRXPT_1, except for the lack of TRXPT_1's second exon. From our analysis, TRXPT_3 and TRXPT_4 seem to have transcription start sites (TSS) downstream of the TSS of TRXPT_1 and TRXPT_2; however, given that studies in MAdVs show that E1 mRNAs share a common 5' and 3' positions, only differing from each other regarding the introns [@Zhao2014], it is likely that TRXPT_3 and TRXPT_4 are truncated, and the TSS just like the transcription termination site (TTS) are identical for all E1 transcripts. Regardless of the TSS considered for TRXPT_3, the coding potential remains unaffected. Its 5'-most CDS, beginning at 1965 and sharing the same STC as TRXPT_1 and TRXPT_2, produces a 13.1 kDa, 115 residue protein. This CDS (ORF4) was predicted in an earlier study [@Pitcovski1998] but was excluded in later studies [@Beach2009a; @Davison2003], but our data suggests it is a bona fide ORF. The coding potential of TRXPT_4 is affected by the TSS considered; if we consider its unmodified TSS, then its coding potential is the same as TRXPT_3 (ORF4 as the first CDS and Hyd as second CDS if the first SSC is skipped). However, if we assume that TRXPT_4 shares the same TSS as TRXPT_1, then the 5'-most CDS is a distinct multi-exonic 15.9 kDa, 143 aa protein with the same SSC as TRXPT_1 and TRXPT_2 but with a unique STC. All splice junctions of the transcripts in this region (except the junction for TRXPT_4) have been validated by cloning and Sanger sequencing of cDNA (__Figure 5b__; __supplementary PCR methods__). Finally, during our validation of TRXPT_2, ORF1 was present on the agarose gel and Sanger sequencing results as a bona fide transcript (__supplementary PCR methods__). This is corroborated by 3' Rapid Amplification of cDNA Ends (3'RACE) experiment, which shows transcripts spanning the entire ORF1 and Hyd ORFs without any splicing. The 5'-most CDS of this transcript would encode ORF1. Given that the SSC of the predicted ORF1 is in-frame but downstream of TRXPT_2 SSC, it suggests that the predicted ORF1 CDS is truncated; it shares the same SSC as TRXPT_2, but has a unique STC, albeit it has the same TTS.

**Early Region 2 (E2) and Intermediate Region (IM) transcripts**. The E2 TU expressed on the reverse strand, is subdivided into E2A and E2B and encodes three classical AdV proteins: pTP and Ad-pol (E2B proteins), and DBP (E2A protein) [####]. Unlike MAdV where two promoters (E2-early and E2-late) were discovered [###], we discovered only a single promoter from which both E2A and E2B transcription is initiated. However, similar to MAdVs, E2A and E2B transcripts have distinct TTSs; the E2B transcripts sharing the TTS of the IVa2 transcript of the IM region [####] (__Figure 6__). ~~Analysis of the coding potential of the transcripts; include the variant of DBP from 3'RACE data~~. 

The IM region is a single-transcript TU, encoding a single classical protein, IVa2. The promoter expressing this single transcript (TRXPT_5) is embedded in E2B region and shares a TTS with E2B transcripts [@Donovan2020; @Zhao2014]. TRXPT_5 is a two-exon transcript located on the reverse strand spliced at 3447-3615. The first exon is an untranslated region (UTR), except the last 2 nucleotides, which connect with the first nucleotide of second exon to form the 5'-most SSC. This first SSC is 4 codons upstream and in-frame of the predicted IVa2 SSC. Regardless of the SSC considered, the encoded protein (IVa2) is largely unaffected. Except for the four extra residues at the N-terminus (considering the 5'-most SSC), the entire protein sequence is identical. 

The splice junction of ~~TRXPT_5 add trxpts from E2~~ were confirmed by cloning and Sanger sequencing of cDNA (__supplementary PCR methods__).

**Early Region 3 (E3) transcripts**.

**Early Region 4 (E4) transcripts**. This transcription unit (TU) is the found at the tail-end (3'-end) of the genome, on the reverse strand. Based on nucleotide position, ORF7 and ORF8 were predicted in this region [@Davison2003]; however, as ORF7 is neither on the same strand as ORF8 nor transcribed from a promoter in the E4 region, only ORF8 can legitimately be classified as a transcript in this TU. This is corroborated by our RNA-seq data, as only one transcript was identified in this region on the reverse strand (__Figure 8__). The transcript (TRXPT_28) spans 25192-26247 and is spliced at 25701-26055; hence, a two-exon transcript. The second exon fully matches the predicted ORF8 with 12 extra base pairs at the 3'-end; however, the encoded protein is an exact match. There is a SSC in the first exon at position 26246 (second nucleotide of the transcript). The encoded protein from this SSC is in-frame with the SSC of ORF8 in the second exon; hence, the C-terminus of this longer protein (26.4 kDa, 229 aa) would be the identical to the predicted ORF8 protein. The splice junction of TRXPT_28 was validated by cloning and Sanger sequencing of cDNA (__supplementary PCR methods__).

**Major Late Promoter Region (MLP) transcripts**.

\newpage

## MATERIALS AND METHODS   
### Cell culture and THEV Infection   
The Turkey B-cell line (MDTC-RP19, ATCC CRL-8135) was grown as suspension cultures in 1:1 complete Leibovitz's L-15/McCoy's 5A medium with 10% fetal bovine serum (FBS), 20% chicken serum (ChS), 5% tryptose phosphate broth (TPB), and 1% antibiotics solution (100 U/mL Penicillin and 100ug/mL Streptomycin), at 41^o^C in a humidified atmosphere with 5% CO~2~. Infected cells were maintained in 1:1 serum-reduced Leibovitz's L15/McCoy's 5A media (SRLM) with 2.5% FBS, 5% ChS, 1.2% TPB, and  1% antibiotics solution (100 U/mL Penicillin and 100ug/mL Streptomycin). A commercially available HE vaccine was purchased from Hygieia Biological Labs as a source of THEV-A (VAS strain). The stock virus was titrated using an in-house qPCR assay with titer expressed as genome copy number(GCN)/mL, similar to Mahshoub *et al*[@Mahsoub2017] with modifications. Cells were infected at a multiplicity of infection (MOI) of 100 GCN/cell and samples in triplicates were harvested at 4-, 12-, 24-, and 72-h.p.i for RNA-seq. The infection was repeated but samples in triplicates were harvested at 12-, 24-, 36-, 48-, and 72-h.p.i for PCR validation of novel splice sites.

### RNA extraction and Sequencing   
Total RNA was extracted from infected cells using Thermofishers' RNAqueous™-4PCR Total RNA Isolation Kit `(#AM1914)` as per manufacturer's instructions. An agarose gel electrophoresis was performed to check RNA integrity. The RNA quantity and purity was initially assessed using `nanodrop`, and RNA was used only if the A260/A280 ratio was 2.0 ± 0.05 and the A260/A230 ratio was >2 and <2.2. Extracted total RNA samples were sent to `LC Sciences, Houston TX` for poly-A-tailed mRNA sequencing where RNA integrity was checked with `Agilent Technologies 2100 Bioanalyzer High Sensitivity DNA Chip` and poly(A) RNA-seq library was prepared following `Illumina's TruSeq-stranded-mRNA sample preparation protocol`. Paired-end sequencing was performed on `Illumina's NovaSeq 6000 sequencing system`.

### Computational Analysis of RNA Sequencing Data: Mapping and Transcript characterization
Analysis of our sequence reads were analyzed following a well established protocol described by Pertea _et al_ [@Pertea2016], using `SNAKEMAKE 7.24.0` to drive the pipeline. Briefly, sequencing reads were trimmed with the `FastQC - version 0.11.9` [@FastQC] program to achieve an overall `Mean Sequence Quality (Phred Score)` of 36. Trimmed reads were mapped to the complete sequence of avirulent turkey hemorrhagic enteritis virus strain Virginia (https://www.ncbi.nlm.nih.gov/nuccore/AY849321.1/) and *Meleagris gallopavo* (https://www.ncbi.nlm.nih.gov/genome/?term=Meleagris+gallopavo) using `Hisat2 - version 2.2.1` [@Pertea2016] with default settings without relying on known splice sites. The generated `BAM` files from each infection time-point were filtered for reads mapping to the THEV genome and fed into `StringTie - version 2.2.1` [@Pertea2016] using a `gff3` file from NCBI containing the predicted ORFs of THEV as a guide. A custom script was used to consolidate all transcripts from all time-points without redundancy, generating the final transcriptome of THEV.

### Validation of Novel Splice Junctions 
All splice junctions identified in this work are novel except one predicted splice site each for pTP and DBP, which were corroborated in our work. However, these predicted splice junctions have not been experimentally validated hitherto, and we identified additional novel splice junctions beside the predicted junctions, giving a more complete picture of the transcripts.

The novel splice junctions after consolidating all transcripts with `StringTie` which we validated by PCR and Sanger Sequencing are shown in ~~Table###1~~. We designed primers that crossed a range of novel exon–exon boundaries for each specific transcript in a transcription unit with their respective universal primers (~~supplementary, PCR methods~~). Each forward primer contained a KpnI restriction site and reverse primers, an XbaI site. After first-strand cDNA synthesis with SuperScript™ III First-Strand Synthesis System (ThermoFisher SCIENTIFIC), these primers were used in a targeted PCR experiment, the PCR products were analysed on Agarose gels, cloned by traditional restriction enzyme method and Sanger sequenced to validate these splice junctions at the sequence level.

### 3' Rapid Amplification of cDNA Ends (3'RACE)  

\newpage

## DISCUSSION/CONCLUSIONS
In the original study where the ORFs of THEV were predicted, ORF4 was predicted in the E1 region spanning the Hyd gene. However, later studies predicted and preferred Hyd instead of ORF4; hence, the current prediction map. However, this study shows that while both Hyd and ORF4 may be both expressed, ORF4 is most likely the bona fide gene.

TRXPT_2 and ORF1 are isoforms
\newpage

## SCRIPTS AND SUPPLEMENTARY MATERIALS    

### DATA AVAILABILITY    

### CODE AVAILABILITY    
All the code/scripts written for analysis of the data is available on github (linkXXXXXX)
\newpage

## ACKNOWLEDGMENTS  
LC Sciences
`Eton Bioscience, Inc, San Diego, CA`
\newpage

## REFERENCES     
\setlength{\parindent}{-0.25in}
\setlength{\leftskip}{0.25in}
\noindent
<div id="refs"></div>
\setlength{\parindent}{0in}
\setlength{\leftskip}{0in}
\newpage

## TABLES AND FIGURES     
![](results/r/figures/thev_orf_map.png)
**Figure 1. _Genomic map of THEV avirulent strain_**. The central horizontal line represents the double-stranded DNA marked at 5kb intervals as white line breaks. Blocks represent viral genes. Blocks above the DNA line are transcribed rightward, those below are transcribed leftward. pTP, DBP and 33K predicted to be spliced are shown as having tails. Shaded regions indicate regions containing "genus-specific" genes (colored red). Genes colored in blue are "genus-common".  Gene colored in light green is conserved in all but Atadenoviruses. The UXP (light blue) is an incomplete gene present in almost all AdVs. Regions comprising the different transcription units are labelled at the bottom (E1, E2A, E2B, E3, E4, and IM); the unlabeled regions comprise the MLTU.
\newpage

```{r}
#| tab.id: NULL
#| tab.cap.pre: ""
#| tab.cap.sep: ""

set_flextable_defaults(font.size = 8)
flextable(tab1) %>% theme_zebra() %>%
  flextable::set_table_properties(layout = "fixed", width = 0.95,
                       opts_pdf = list(caption_repeat = F)) %>%
  flextable::font(part = "all", fontname = "Avenir") %>%
  flextable::width(., j = ~ Metric, width = 1.5) %>%
  flextable::set_caption("Overview of sequencing results",
              style = "Table Caption") %>%
  flextable::bold(part = "body", j = ~ Metric)

```
\newpage

![](results/r/figures/overlay_alltimes.png)
![](results/r/figures/thev_growth_curve.png)
**Figure 2. a) _Per base coverage of sequence reads mapping to THEV genome by time point_**. The pileup of mRNA reads mapping to THEV genome at the base-pair level for each indicated time point show remarkable difference in terms of quantities. There is a dramatic increase of mean coverage/depth from `r dep_4` at 4 h.p.i to `r dep_72` at 72 h.p.i, strongly demonstrating an active infection. Unexpectedly, the pileup of reads seems consistently skewed over similar regions of the genome. We could speculate that the temporal gene expression regulation of THEV is different from MAdVs or this could simply mean that the infection was not well synchronized. However, the relative proportions over these similar regions shows some variation over time. **b) _One-step growth of THEV (VAS vaccine strain) in MDTC-RP19 cell line_**. After infecting cells at an MOI of 100 GCN/cell, triplicates of harvested infected cells were quantified with an in-house qPCR assay measuring the total copies of THEV genome. There is no discernible increase in virus titer up 12 h.p.i, after which there is a steady increase in virus titer is measured. The virus titer expands exponentially beginning from 48 h.p.i, increasing by orders of magnitude before reaching a plateau at 120 h.p.i, probably due to high cell death. GCN: genome copy number.

\newpage
![](results/r/figures/thev_spliced_map.png)
![](results/r/figures/thev_patched_timepoints_spliced_map.png)
**Figure 3. a) __Full transcriptome of THEV__**. THEV transcripts assembled from all time points by `StringTie` are unified forming this final transcriptome (splicing map). Transcripts belonging to the same transcription unit (TU) are located in close proximity on the genome and are color coded and labeled in this figure as such. The organization of TUs in the THEV genome is unsurprisingly similar to MAdVs; however, the MAdV genome shows significantly more transcripts. The TUs are color coded: E1 transcripts - red, E2 - black, E3 - dark grey, E4 - green, MLP - blue. Predicted ORFs are also indicated here, colored light grey. **b) __THEV transcripts identified at given time points__**. Transcripts are color coded as explained in **a**. 

\newpage

```{r}
#| tab.id: NULL
#| tab.cap.pre: ""
#| tab.cap.sep: ""
set_flextable_defaults(font.size = 8)
flextable(sig_12_juncs) %>% theme_zebra() %>%
  flextable::set_table_properties(layout = "fixed", width = 1,
                       opts_pdf = list(caption_repeat = F)) %>%
  flextable::font(part = "all", fontname = "Avenir") %>%
  flextable::set_caption("Table 2a: Most abundant splice junctions at 12h.p.i",
              style = "Table Caption") %>%
  flextable::align(align = "center", part = "all") %>%
  flextable::width(., width = 0.7) %>%
  flextable::width(., j = ~ Reads_Percentage, width = 1)
```
\newpage

```{r}
#| tab.id: NULL
set_flextable_defaults(font.size = 8)
flextable(sig_24_juncs) %>% theme_zebra() %>%
  flextable::set_table_properties(layout = "fixed", width = 1,
                       opts_pdf = list(caption_repeat = F)) %>%
  flextable::font(part = "all", fontname = "Avenir") %>%
  flextable::set_caption("Table 2b: Most abundant splice junctions at 24h.p.i",
              style = "Table Caption") %>%
  flextable::align(align = "center", part = "all") %>%
  flextable::width(., width = 0.7) %>%
  flextable::width(., j = ~ Reads_Percentage, width = 1)
```
\newpage

```{r}
#| tab.id: NULL
set_flextable_defaults(font.size = 8)
flextable(sig_72_juncs) %>% theme_zebra() %>%
  flextable::set_table_properties(layout = "fixed", width = 1,
                       opts_pdf = list(caption_repeat = F)) %>%
  flextable::font(part = "all", fontname = "Avenir") %>%
  flextable::set_caption("Table 2c: Most abundant splice junctions at 72h.p.i",
              style = "Table Caption") %>%
  flextable::align(align = "center", part = "all") %>%
  flextable::width(., width = 0.7) %>%
  flextable::width(., j = ~ Reads_Percentage, width = 1)
```
\newpage

![](results/r/figures/trxpt_fpkm_percent_abund.png)

![](results/r/figures/region_fpkm_percent_abund.png)
**Figure 4 Changes in splicing and expression profile of THEV over time.** 
**a)** Expression levels of transcripts over time.
**b)** Expression levels of transcripts by region over time.

\newpage
![](results/r/figures/junc_abundances.png)
**c)** Relative abundances of all splice junctions over time. **d)** Relative abundances of junctions in final transcriptome.


```{r region-by-region_e1_figure, fig.height=16, fig.width=18, warning=FALSE}
set_flextable_defaults(font.size = 7)
ft <- flextable(reg_e1_brkdown,
                col_keys = c("trxpt_id", "start", "end", "splice_site", "intron_len",
                             "strand", "4hpi", "12hpi", "24hpi", "72hpi", "status")) %>%
  theme_zebra() %>%
  flextable::set_table_properties(layout = "fixed",
                       opts_pdf = list(caption_repeat = F)) %>%
  flextable::set_header_labels(start = "Start", end = "End",
                               splice_site = "Splice Site",
                               strand = "Strand", trxpt_id = "Transcript ID",
                               intron_len = "Intron Length", "4hpi" = "4h.p.i",
                               "12hpi" = "12h.p.i", "24hpi" = "24h.p.i",
                               "72hpi" = "72h.p.i", "status" = "Junction Status") %>%
  flextable::align(align = "center", part = "all") %>%
  add_header_row(., values = c("", "Splice Junction", "", "Junction Reads", ""),
                 colwidths = c(1, 4, 1, 4, 1)) %>%
  flextable::vline(j = c(1, 5, 6, 10), border = fp_border_default(color = "grey",
                                                                  width = 1.5)) %>%
  footnote(., i = 2, j = 11, value = as_paragraph(c("Not validated for TRXPT_4")),
           ref_symbols = c("*"), part = "body")

brkdwn_e1_trxtps / plot_spacer()/ gen_grob(ft, fit = "auto", scaling = "full") +
  plot_layout(heights = c(2, 0.05, 2)) +
  plot_annotation(tag_levels = "A") &
  theme(plot.tag = element_text(size = 22, face = "bold"))

```

**Figure 5: E1 region transcripts. a)** The splice map of the E1 transcription unit. Exons are depicted as boxes connected by introns (dotted lines). Transcripts from RNA-seq data are colored red, predicted ORFs are colored grey, and the previously annotated ORF4 is colored black. Each transcript or ORF is labelled with its name to the right. The start codon (SSC) and stop codon (STC) of the 5'-most CDS of each transcript is indicated with the nucleotide position in brackets. The region of the virus is depicted at the bottom as a black line with labels of the nucleotide positions for reference **b)** The sequence reads covering the splice junctions are indicated with information about their validation status using cloning and Sanger sequencing.

```{r region-by-region_e2ANDim_figure, fig.height=16, fig.width=18, warning=FALSE}
set_flextable_defaults(font.size = 7)
ft <- flextable(reg_e2_brkdown,
                col_keys = c("trxpt_id", "start", "end", "splice_site", "intron_len",
                             "strand", "region", "4hpi", "12hpi", "24hpi", "72hpi", "status")) %>%
  theme_zebra() %>%
  flextable::set_table_properties(layout = "fixed",
                       opts_pdf = list(caption_repeat = F)) %>%
  flextable::set_header_labels(start = "Start", end = "End",
                               splice_site = "Splice Site",
                               strand = "Strand", trxpt_id = "Transcript ID",
                               intron_len = "Intron Length", "4hpi" = "4h.p.i",
                               "12hpi" = "12h.p.i", "24hpi" = "24h.p.i",
                               "72hpi" = "72h.p.i", "status" = "Junction Status") %>%
  flextable::align(align = "center", part = "all") %>%
  add_header_row(., values = c("", "Splice Junction", "", "Junction Reads", ""),
                 colwidths = c(1, 4, 2, 4, 1)) %>%
  flextable::vline(j = c(1, 5, 7, 11), border = fp_border_default(color = "grey",
                                                                  width = 1.5)) %>%
  footnote(., i = 9, j = 12, value = as_paragraph(c("Truncated transcript")),
           ref_symbols = c("*"), part = "body")

brkdwn_e2_trxtps / plot_spacer()/ gen_grob(ft, fit = "auto", scaling = "full") +
  plot_layout(heights = c(1, 0.05, 2))

```

**Figure 6: E2 and IM region transcripts. a)** The splice map of the E1 and IM transcription units. Exons are depicted as boxes connected by introns (dotted lines). Red transcripts are generated from RNA-seq data and predicted ORFs are colored grey. Each transcript or ORF is labelled with its name to the right. The start codon (SSC) and stop codon (STC) of the 5'-most CDS of each transcript is indicated with the nucleotide position in brackets. The region of the virus is depicted at the bottom as a black line with labels of the nucleotide positions for reference **b)** The sequence reads covering the splice junctions are indicated with information about their validation status using cloning and Sanger sequencing.


```{r region-by-region_e4_figure, fig.height=5, fig.width=7, warning=FALSE}
reg_e4_brkdown <- reg_e4_brkdown %>%
  mutate(splice_site = "GT-AG",
         `4hpi` = as.numeric(`4hpi`), `12hpi` = as.numeric(`12hpi`),
         `24hpi` = as.numeric(`24hpi`), `72hpi` = as.numeric(`72hpi`)) %>%
  select(-c(coding_potential, region)) %>%
  group_by(trxpt_id, start, end, splice_site, intron_len, strand, status) %>%
  reframe("4hpi" = sum(`4hpi`),
          "12hpi" = sum(`12hpi`),
          "24hpi" = sum(`24hpi`),
          "72hpi" = sum(`72hpi`)) %>%
  select(trxpt_id, start, end, splice_site, intron_len, strand, "4hpi", "12hpi",
         "24hpi", "72hpi", status)

set_flextable_defaults(font.size = 7)
ft <- flextable(reg_e4_brkdown,
                col_keys = c("trxpt_id", "start", "end", "splice_site", "intron_len",
                             "strand", "4hpi", "12hpi", "24hpi", "72hpi", "status")) %>%
  theme_zebra() %>%
  flextable::set_table_properties(layout = "fixed",
                       opts_pdf = list(caption_repeat = F)) %>%
  flextable::set_header_labels(start = "Start", end = "End",
                               splice_site = "Splice Site",
                               strand = "Strand", trxpt_id = "Transcript ID",
                               intron_len = "Intron Length", "4hpi" = "4h.p.i",
                               "12hpi" = "12h.p.i", "24hpi" = "24h.p.i",
                               "72hpi" = "72h.p.i", "status" = "Junction Status") %>%
  flextable::align(align = "center", part = "all") %>%
  add_header_row(., values = c("", "Splice Junction", "", "Junction Reads", ""),
                 colwidths = c(1, 4, 1, 4, 1)) %>%
  flextable::vline(j = c(1, 5, 6, 10), border = fp_border_default(color = "grey",
                                                                  width = 1.5))

brkdwn_e4_trxtps / plot_spacer()/ gen_grob(ft, fit = "auto", scaling = "full") +
  plot_layout(heights = c(1, 0.01, 1)) +
  plot_annotation(tag_levels = "A") &
  theme(plot.tag = element_text(size = 18, face = "bold"))

```

**Figure 8:  E4 region transcripts. a)** The splice map of the E4 transcription unit. Exons are depicted as boxes connected by introns (dotted lines). The transcript from RNA-seq data is colored red and the predicted ORF, grey. The transcript and ORF are labelled with their names to the right. The start codon (SSC) and stop codon (STC) of the 5'-most CDS is indicated with the nucleotide position in brackets. The region of the virus is depicted at the bottom as a black line with labels of the nucleotide positions for reference **b)** The sequence reads covering the splice junction are indicated.

```{r region-by-region_im_figure, fig.height=5, fig.width=10, warning=FALSE, include=FALSE}
reg_im_brkdown <- reg_im_brkdown %>%
  mutate(trxpt_id = "TRXPT_5")

set_flextable_defaults(font.size = 7)
ft <- flextable(reg_im_brkdown,
                col_keys = c("trxpt_id", "start", "end", "splice_site", "intron_len",
                             "strand", "4hpi", "12hpi", "24hpi", "72hpi", "status")) %>%
  theme_zebra() %>%
  flextable::set_table_properties(layout = "fixed",
                       opts_pdf = list(caption_repeat = F)) %>%
  flextable::set_header_labels(start = "Start", end = "End",
                               splice_site = "Splice Site",
                               strand = "Strand", trxpt_id = "Transcript ID",
                               intron_len = "Intron Length", "4hpi" = "4h.p.i",
                               "12hpi" = "12h.p.i", "24hpi" = "24h.p.i",
                               "72hpi" = "72h.p.i", "status" = "Junction Status") %>%
  flextable::align(align = "center", part = "all") %>%
  add_header_row(., values = c("", "Splice Junction", "", "Junction Reads", ""),
                 colwidths = c(1, 4, 1, 4, 1)) %>%
  flextable::vline(j = c(1, 5, 6, 10), border = fp_border_default(color = "grey",
                                                                  width = 1.5))

brkdwn_im_trxtps / plot_spacer()/ gen_grob(ft, fit = "auto", scaling = "full") +
  plot_layout(heights = c(1, 0.01, 1)) +
  plot_annotation(tag_levels = "A") &
  theme(plot.tag = element_text(size = 18, face = "bold"))

```

<!-- **Figure 9 IM region transcripts. a)** The splice map of the IM transcription unit. **b)** The sequence reads covering the splice junction are indicated. -->


