---
title: Characterizing the Splice Map of Turkey Hemorrhagic Enteritis Virus
output:
  officedown::rdocx_document:
    reference_docx: style_word_output.docx
  pdf_document:
    keep_tex: true
geometry: margin=1in
font-size: 11pt
header-includes:
  - \usepackage{helvet}
  - \renewcommand*\familydefault{\sfdefault}
  - \usepackage{setspace}
  - \doublespacing
  - \usepackage[left]{lineno}
  - \linenumbers
  - \usepackage{booktabs}
  - \usepackage{longtable}
  - \usepackage{array}
  - \usepackage{multirow}
  - \usepackage{wrapfig}
  - \usepackage{float}
  - \usepackage{placeins}
  - \usepackage{colortbl}
  - \usepackage{pdflscape}
  - \usepackage{tabu}
  - \usepackage{threeparttable}
  - \usepackage{threeparttablex}
  - \usepackage[normalem]{ulem}
  - \usepackage{makecell}
  - \usepackage{xcolor}
  - \usepackage{caption}
  - \usepackage[labelformat=empty]{caption}
bibliography: transcriptome_refs.bib
csl: asm.csl
editor_options:
  chunck_output_type: console
  markdown: 
    wrap: sentence
---

\captionsetup[table]{labelformat=empty}

\vspace{5mm}

Abraham Quaye${^\dagger}$^,a^, Brett E. Pickett^a^, Joel S. Griffitts^a^, Bradford K. Berges^a^, Brian D. Poole^a,\*^

\vspace{5mm}

^a^Department of Microbiology and Molecular Biology, Brigham Young University\
${^\dagger}$First-author\
^\*^Corresponding Author

\vspace{5mm}

**Corresponding Author Information**\
[brian_poole\@byu.edu](mailto:brian_poole@byu.edu)\
Department of Microbiology and Molecular Biology,\
4007 Life Sciences Building (LSB),\
Brigham Young University,\
Provo, Utah\
\newpage

```{r call_libs, echo=FALSE,include=FALSE, message=FALSE, warning=FALSE}
library(knitr)
library(tidyverse)
library(tibble)
library(scales)
library(flextable)
library(english)
library(rticles)
opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
knitr::opts_chunk$set(tab.cap.pre = "", tab.cap.sep = "")
```

```{r inline_values, echo=FALSE,include=FALSE, message=FALSE, warning=FALSE}

# extract paired-end sequencing read length
bam_info <- read_tsv("results/sgseq/bam_extracted_sampleData.tsv",
                     show_col_types = F) 
seq_len <- bam_info %>% select(read_length) %>% slice_head() %>% pull()


## load in variables to be used inline 
source("scripts/r/bam_file_analysis.R")
source("scripts/r/thev_splicing_fullMap.R")
source("scripts/r/abundance_analyses.R")
source("scripts/r/reg_by_reg_plots.R")


cov_thev <- all_covs %>% filter(organism == "thev")

cov_4hr <- cov_thev %>% filter(timepoint == "4hpi") %>%
              select(total_mapped) %>% pull() %>% as.numeric(.)
cov_72hr <- cov_thev %>% filter(timepoint == "72hpi") %>%
              select(total_mapped) %>% pull() %>% as.numeric(.)
cov_72hr <- round(cov_72hr / 1e6, 1)

total_mapped <- round(sum(as.numeric(cov_thev$total_mapped)) / 1e6, 1)

avg_tot_count <- round(mean(total_reads$total_reads) / 1e6, 1)


### inline code for figure 2: depth graph
dep_4 <- as.numeric(tab1$"4h.p.i"[4])
dep_72 <- as.numeric(str_replace(tab1$"72h.p.i"[4], ",", "")) %>% comma(.)

## inline code of junction stats
tot_jun_4 <- tab1$`4h.p.i`[5]
tot_jun_72 <- as.character(tab1$`72h.p.i`[5])
mean_juncreads_4 <- tab1$`4h.p.i`[7]
mean_juncreads_72 <- round(as.numeric(tab1$`72h.p.i`[7]), 1)
max_junread72 <- bulk_junc_stats %>% filter(timepoint == "72hpi") %>%
  pull(read_count) %>% max(.) %>% comma(.)

most_sig_12 <- bulk_junc_stats %>% filter(timepoint == "12hpi", region == "MLP") %>% pull(read_count) %>% sum()/as.numeric(tab1$`12h.p.i`[6])*100

most_sig_24 <- bulk_junc_stats %>% filter(timepoint == "24hpi", region == "MLP") %>% pull(read_count) %>% sum()/as.numeric(str_remove(tab1$`24h.p.i`[6], ","))*100

most_sig_72 <- bulk_junc_stats %>% filter(timepoint == "72hpi", region == "MLP") %>% pull(read_count) %>% sum()/as.numeric(tab1$`72h.p.i`[6])*100
```

## ABSTRACT   
**Background:** Hemorrhagic enteritis, caused by _Turkey Hemorrhagic Enteritis Virus (THEV)_, is a disease affecting turkey poults characterized by immunosuppression and bloody diarrhea. An avirulent THEV strain that retains the immunosuppressive ability is used as a live vaccine. Characterizing the splice map of THEV is an essential step that would allow studies of individual genes mediating its immunosuppressive functions. We used RNA sequencing to characterize the splice map of THEV for the first time, providing key insights into the THEV gene expression and mRNA structures.   
**Methods:** After infecting a turkey B-cell line with the vaccine strain, samples in triplicates were collected at 4-, 12-, 24-, and 72-hours post-infection. Total RNA was extracted, and poly-A-tailed mRNA sequenced. Reads were mapped to the THEV genome after trimming and transcripts assembled with StringTie. We performed PCR of THEV cDNA, cloned the PCR products, and used Sanger sequencing to validate all identified splice junctions.   
**Results:** Researchers previously annotated the THEV genome as encoding 23 open reading frames (ORFs). We identified 29 spliced transcripts from our RNA sequencing data, all containing novel exons although some exons matched some previously annotated ORFs. The three annotated splice junctions were also corroborated by our data. During validation we identified five additional unique transcripts, a subset of which were further validated by 3' rapid amplification of cDNA ends (3' RACE). Thus, we report that the genome of THEV contains 34 transcripts with the coding capacity for all annotated ORFs. However, we found six of the previously annotated ORFs to be truncated ORFs on the basis of the identification of an in-frame upstream start codon or the detection of additional coding exons. We also identified three of the annotated ORFs with longer or shorter isoforms, and seven novel unannotated ORFs that could potentially be translated; although it is beyond the scope of this manuscript to investigate whether they are translated.   
**Conclusions:** Similar to human adenoviruses, all THEV transcripts are spliced and organized into five transcription units under the control of their cognate promoters. The genes are expressed under temporal regulation and THEV also produces multiple distinctly spliced transcripts that code for the same protein. Studies of the newly identified potential proteins should be urgently performed as these proteins may have roles in THEV-induced immunosuppression. Also, knowing the splicing of THEV genes should be invaluable to future research focusing of studying THEV genes, as this will allow accurate cloning of the mRNAs. 

## KEY WORDS
Alternative splicing, Turkey hemorrhagic enteritis virus, Adenovirus, Transcriptome, RNA sequencing.
\newpage

## BACKGROUND

Adenoviruses (AdVs) are non-enveloped icosahedral-shaped DNA viruses, causing infection in virtually all types of vertebrates studied to date. Their double-stranded linear DNA genomes range between 26 and 45kb in size, producing a broad repertoire of transcripts via highly complex alternative splicing patterns [@Davison2003; @Harrach2008]. The AdV genome is one of the most optimally economized; both the forward and reverse DNA strands harbor protein-coding genes, making it highly gene-dense. There are 16 genes termed "genus-common" that are homologous in all AdVs, presumably inherited from a common ancestor. All other genes are termed "genus-specific". The genus-specific genes tend to be located at the termini of the genome while genus-common genes are usually towards the center of the genome [@Davison2003]. This pattern is also observed in *Poxviridae* and *Herpesviridae*, which also have linear DNA genomes [@Davison2003; @Upton2003; @McGeoch1999]. The family *Adenoviridae* consists of five genera: *Mastadenovirus* (MAdV), *Aviadenovirus*, *Atadenovirus*, *Ichtadenovirus*, and *Siadenovirus* (SiAdV) to which turkey adenovirus 3 also called turkey hemorrhagic enteritis virus (THEV) belongs [@Harrach2011; @Kovács2011; @Davison2000; @Kovács2010; @Katoh2009; @Guimet2016]. Members of SiAdV have the smallest genome size (\~26 kb) and gene content of all known AdVs, with five genus-specific genes of undefined functions (see **Figure 1**) [@Harrach2008; @Kovács2011; @Davison2003].

Virulent THEV strains (THEV-V) and avirulent strains (THEV-A) of THEV both infect turkeys, with THEV-V causing hemorrhagic enteritis (HE), a debilitating acute disease predominantly affecting turkey poults characterized by immunosuppression, intestinal lesions leading to bloody diarrhea, and up to 80% mortality [@Beach2006; @Beach2009a; @Gross1967; @Harrach2008]. While the current vaccine strain (a THEV-A called Virginia Avirulent Strain [VAS]) has proven effective at preventing HE in turkey poults, it still retains its immunosuppressive ability. Thus, vaccinated birds are rendered more susceptible to opportunistic infections and death than unvaccinated birds leading to substantial economic losses [@Beach2006; @Rautenschlein2000; @Larsen1985; @Dhama2017]. To eliminate the immunosuppressive effect of the vaccine strain, a thorough investigation of the culprit viral genes mediating this phenomenon is essential. However, the transcriptome (splicing and gene expression patterns) of THEV has not been characterized, making an investigation of specific immunosuppressive viral genes impractical.

A myriad of studies have elucidated the AdV transcriptome in fine detail [@Donovan2020; @Zhao2014]. However, a large preponderance of studies focus on MAdVs -- specifically human AdVs. Thus, most of the current AdV gene expression and replication knowledge is based on MAdV studies, which is generalized for all other AdVs [@Guimet2016; @Wolfrum2013]. MAdV transcription is temporally regulated; their genes are categorized into five early transcription units (E1A, E1B, E2, E3, and E4), two intermediate (IM) units (pIX and IVa2), and one major late transcription unit (MLTU or major late promoter [MLP] region), which generates five families of late mRNAs (L1-L5) based on the polyadenylation site. An additional gene (UXP or U exon) is located on the reverse strand. The early genes encode non-structural proteins such as enzymes or host-cell modulating proteins, primarily involved in DNA replication, or providing the necessary intracellular niche for optimal replication while late genes encode structural proteins that act as capsid proteins, promote virion assembly, or direct genome packaging. The immediate early genes E1A are expressed first, followed by the delayed early genes, E1B, E2, E3 and E4. Then the intermediate early genes, IVa2 and pIX are expressed followed by the late genes [@Zhao2014; @Donovan2020; @Guimet2016]. It is noteworthy that the MLP shows basal transcriptional activity during early infection (before DNA replication), with a comparable efficiency to other early viral promoters, but it reaches its maximal activity during late infection (after DNA replication). However, during early infection only a subset of the MLP-derived transcripts are expressed [@Guimet2016]. MAdVs make extensive use of alternative RNA splicing and polyadenylation to produce a very complex array of mRNAs. All but the pIX mRNA undergo at least one splicing event. For instance, the MLTU produces over 20 distinct splice variants all containing three non-coding exons at the 5'-end (collectively known as the tripartite leader; TPL) [@Donovan2020; @Zhao2014]. There is also an alternate three-exon 5' non-coding leader sequence present in varying amounts on a subset of MLTU mRNAs (known as the x-, y-, and z-leaders). Lastly, there is the i-leader exon, which is infrequently included between the second and third TPL exons, and codes for the i-leader protein [@Falvey1983]. Thus, the MLTU produces a complex repertoire of mRNA with diverse 5' untranslated regions (UTRs) spliced onto different 3' coding exons which are grouped into five different 3'-end classes (L1-L5) based on polyadenylation site. Each transcription unit (TU) contains its own promoter driving the expression of the array of mRNA transcripts produced via alternative splicing in the unit [@Zhao2014; @Donovan2020; @Guimet2016]. The promoters are activated at different phases of infection by proteins from previously activated TUs. Paradoxically, the early-to-late phase transition during infection requires the L4 gene products, 22K and 33K, which should only be available after the transition. However, a promoter in the L4 region (L4P) that directs the expression of these two proteins independent of the MLP was found, resolving the paradox [@Morris2010; @Guimet2016; @Donovan2020]. During translation of AdV mRNA, recent studies using long-read direct RNA sequencing strongly suggest the potential usage of secondary start codons; adding to what was already a highly complex system for gene expression [@Donovan2020; @Westergren2021].

High throughput sequencing methods have facilitated the discovery of many novel transcribed regions and splicing isoforms. It is also a very powerful tool to study alternative splicing under different conditions at an unparalleled depth [@Djebali2012; @Zhao2014; @Westergren2021]. In this paper, we use a paired-end deep sequencing experiment to characterize, for the first time, the transcriptome and splicing of THEV (VAS vaccine strain) during different phases of the infection. Our paired-end sequencing allowed for reading `r seq_len` bp long high quality (mean Phred Score of 36) sequences from each end of cDNA fragments, which were mapped to the genome of THEV.
\newpage

## RESULTS

**Overview of sequencing data and analysis pipeline outputs**   
A prior study by Aboezz *et al*. demonstrated that nearly all THEV transcripts became detectable starting at 4 hours post-infection (hpi), with one replication cycle concluding around 18 hpi [@Aboezz2019]. Consequently, we harvested infected MDTC-RP19 cells (multiplicity of infection (MOI) of 100 genome copy numbers/cell) at 4-, 12-, 24-, and 72-hpi to capture all transcripts within a broad time window. Our paired-end RNA sequencing (RNA-seq) experiment generated an average of `r avg_tot_count` million total reads of `r seq_len` bp length per time-point. These reads were concurrently mapped to both the virus (THEV) and host (*Meleagris gallopavo*) genomes using the Hisat2 [@Pertea2016] reference-based aligner. A total of `r total_mapped` million reads from all time-points mapped to the virus genome, providing comprehensive coverage and leaving no regions unmapped. The mapped reads to the virus genome increased significantly from a scant `r cov_4hr` reads at 4 hpi to `r cov_72hr` million reads at 72 hpi (**Table 1**, **Figure 2A**). From these mapped reads, we identified `r comma(nrow(total_unq_jncs_submit))` unique THEV splice junctions across all time-points, with later time-points exhibiting significantly more sequence reads supporting the splice junctions than earlier time-points. For instance, all `r tot_jun_4` unique junctions at 4 hpi had fewer than 10 supporting reads each, averaging only `r mean_juncreads_4` reads per junction. In contrast, the `r tot_jun_72` unique junctions at 72 hpi averaged `r mean_juncreads_72` reads per junction, with some junctions reaching as high as `r max_junread72` reads. The marked increase in splice junction and mapping reads to the THEV genome over time indicates an active infection and successful viral replication, which is corroborated by our quantitative PCR (qPCR) assay that quantified the total number of viral genome copies over time (**Figure 2B**).

Using StringTie [@Pertea2016], we assembled the data into potential transcripts, guided by the genomic locations of the previously predicted THEV ORFs. In the consolidated transcriptome, a composite of all non-redundant transcripts across all time points, we identified a total of `r nrow(spliced_gtf)-1` novel transcripts. We found that a subset of exons in the viral transcripts match some predicted ORFs exactly, with the majority of the exons being longer and spanning multiple predicted ORFs (**Figure 3**).

We then validated the splice junctions in all transcripts by PCR amplification of viral cDNA, cloning, and Sanger sequencing (**Supplementary PCR methods**). During validation, we identified five additional transcripts, some of which were further validated by 3' Rapid Amplification of cDNA Ends (3' RACE) data. The complete list of unique splice junctions mapped to the THEV genome has been submitted to the National Center for Biotechnology Information Gene Expression Omnibus (<http://www.ncbi.nlm.nih.gov/geo>) under accession number GSE254416.

**Changes in THEV splicing profile over time**  
AdV gene expression is subject to meticulous temporal regulation, with each promoter typically generating one or a few pre-mRNAs. These pre-mRNAs undergo alternative splicing to produce a diverse array of mature mRNAs. To assess the temporal activity of each promoter, we utilized StringTie and Ballgown (a tool for statistical analysis of assembled transcriptomes) [@Ballgown]. These tools estimated the normalized expression levels of all transcripts at each time point, measured in Fragments Per Kilobase of transcript per Million mapped reads (FPKM) units. At 4 hpi, we counted very few unique splice junctions, reads, and transcripts; hence, this time-point was excluded from this analysis.

```{r temporal transcript expression levels}
abn_each_12 <- t_expr_each %>% filter(trxpt_id == "E2: TRXPT_21", timepoint == "12h.p.i") %>% pull(fpkm_trxpt_percent)
```

Examining individual mRNAs, TRXPT_21 – from the E2 region – was the most significantly expressed at 12 hpi, constituting `r abn_each_12`% of the total expression of all transcripts. Transcripts in the E3 and E4 regions also contributed substantial proportions, along with some MLP region transcripts. The later time points were dominated by the MLP region transcripts –- TRXPT_10 and TRXPT_14 were the most abundantly expressed at 24 and 72 hpi, respectively (**Figure 4A**). Our analysis of the FPKM values of transcripts per region/TU revealed a similar pattern: the E2 region was the most abundantly expressed at 12 hpi, after which the MLP region assumed dominance (**Figure 4B**).

Next, we estimated the relative abundances of all splice junctions at each time point using the raw reads. Only junctions with a read coverage of at least 1% of the total splice junction reads at the given time point were considered significant and included in **Tables 2a-2c**. At 12 hpi, `r nrow(sig_12_juncs)` junctions met the 1% threshold, predominantly from early regions (E1, E2, E3, and E4), although the MLTU was the single most predominant region overall, constituting `r round(most_sig_12, 1)`% of all the junction reads (**Table 2a** and **Supplementary Table S1a**). The most abundant junctions at 12 hpi remained the most significantly expressed at 24 hpi. However, here, the MLP-derived junctions unsurprisingly became even more predominant overall, accounting for `r round(most_sig_24, 1)`% of all the junction reads counted (**Table 2b** and **Supplementary Table S1b**). At 72 hpi, the trend of increased activity of the MLP continued as expected; at this time, the MLP region junctions were not only the most abundant overall –- accounting for `r round(most_sig_72, 1)`% of all junction reads, -– but also contained the most significantly expressed individual junctions (**Table 2c**, **Supplementary Table S1c** and **Figure 4C**). When we limited this analysis to only junctions in the final transcriptome, we observed the relative abundances of the junctions for each region over time to be similar to the pattern seen with all the junctions included (**Figure 4D**).

Finally, we analyzed splice donor and acceptor site nucleotide usage over time to investigate any peculiarities that THEV may exhibit, generally or over the course of the infection. We found that most splice donor-acceptor sequences were, unsurprisingly, the canonical GU-AG nucleotides. However, the splice acceptor-donor pairing became less specific over time, such that all combinations of nucleotide pairs were eventually detected (**Figure 5**).

**Early Region 1 (E1) transcripts**   
In MAdVs, E1 is the first region transcribed post-viral DNA entry into the host cell nucleus, mediated solely by host transcription machinery [@Zhao2014]. Translated E1 proteins subsequently activate other viral promoters in conjunction with host transcription factors [@Guimet2016]. Despite subdivision into E1a and E1b units in MAdVs, our THEV data does not reflect this. This region is predicted to encode only two ORFs: ORF1 (sialidase) and Hyd (a hydrophobic product with unknown function) in THEV.

We identified `r as.character(english(nrow(filter(spliced_gtf, region == "E1"))))` novel transcripts in this region, containing `r english(nrow(reg_e1_brkdown))` unique splice junctions (**Figure 6**), and encoding `r as.character(english(nrow(filter(spliced_gtf, region == "E1"))))` distinct novel ORFs in addition to Hyd. All transcripts have coding potential (CP) for the Hyd protein as the 3’-most coding sequence if secondary start codon usage is considered [@Donovan2020; @Zhao2014]. Also, all the transcripts have a common transcription termination site (TTS; at position 2325 bp), but TRXPT_1 and TRXPT_2 have an upstream transcription start sites (TSS) to TRXPT_3 and TRXPT_4. Given that E1 mRNAs in MAdVs share a common TTS and TSS, differing only in the internal splicing [@Zhao2014], we consider the upstream TSS (position 54 bp) as the actual TSS for all E1 region transcripts. We also identified the canonical polyadenylation signal (PAS; AAUAAA) in the immediate context of the TTS at position 2323 bp (location of the "U" in the PAS sequence); see **Supplementary Table S2**.

From the 5'-most start codon (SC), TRXPT_1 encodes a multi-exonic novel 17.9 kDa, 160 residue protein (ORF9). TRXPT_2 encodes a two-exon 66.4 kDa, 597 residue novel protein (ORF10), spanning almost the entire predicted ORF1 and Hyd. The intron of TRXPT_2 excludes the C-terminus of ORF1 (including its stop codon) from ORF10 but the SC of ORF10 is 102 bp upstream and in-frame with the predicted SC of ORF1. TRXPT_3, similar to TRXPT_1 but lacking the second exon, encodes a 13.1 kDa, 115 residue protein (ORF4), previously predicted [@Pitcovski1998] but excluded in later annotations [@Beach2009a; @Davison2003]. Our data suggest it is genuinely expressed. Lastly, TRXPT_4 encodes a distinct novel 15.9 kDa, 143 residue protein (ORF11).

The splice junctions of all transcripts in this region, except for TRXPT_4, were validated by cloning of viral cDNA and Sanger sequencing (see **Supplementary PCR methods**). During TRXPT_2 validation, ORF1 was found on the agarose gel (an unspliced band size) and Sanger sequencing results showed it to be a transcribed mRNA (**Supplementary PCR methods**). This was corroborated by our 3' RACE experiment, which showed a transcript (TRXPT_2B) spanning the entire ORF1 and Hyd ORFs without splicing, with a poly-A tail immediately after the E1 TTS. The 5'-most coding sequence (CDS) of this transcript (TRXPT_2B) encodes ORF1. However, TRXPT_2B has an upstream and in-frame SC to the predicted SC of ORF1, suggesting that the predicted ORF1 CDS is truncated -- the expressed ORF1 (eORF1) shares the same SC as ORF10, but has a unique stop codon (STC). See **Supplementary Table S3** for all transcripts and their encoded proteins.

**Early Region 2 (E2) and Intermediate Region (IM) transcripts**   
The E2 TU expressed on the anti-sense strand, is subdivided into E2A and E2B and encodes three classical AdV proteins -- pTP and Ad-pol (E2B proteins), and DBP (E2A protein) -- essential for genome replication [@Donovan2020; @Zhao2014]. Unlike MAdV where two promoters are known [@Donovan2020], we discovered only a single TSS (E2 TSS; 18,751 bp) for both E2A and E2B transcripts in THEV. However, E2A and E2B transcripts have distinct TTSs, with E2B transcripts sharing the TTS of the IVa2 transcript of the IM region similar to MAdVs [@Donovan2020; @Zhao2014] (**Figure 7**).

The E2A ORF, DBP, is one of three THEV ORFs predicted to be spliced from two exons. The corresponding transcript (TRXPT_21) in our data matches the predicted splice junction but includes an additional non-coding exon at the 5'-end (E2-5'UTR). Thus, TRXPT_21 is a three-exon transcript encoding DBP (380 residues, 43.3 kDa) precisely. TRXPT_21 was also corroborated in a 3' RACE experiment. Additionally, from the 3' RACE data, we found a splice variant of TRXPT_21 which retains the second intron, leading to a 2-exon transcript (TRXPT_21B). Although longer, TRXPT_21B encodes a truncated isoform of DBP (tDBP; a 346 residue, 39.3 kDa product) using a downstream in-frame SC but the same STC as DBP. Both TRXPT_21 and TRXPT_21B share a common TTS, seen in our 3' RACE data, located 39 bp downstream of the CDS in an adenine/thymine (A/T)-rich sequence followed by the poly-A tail sequence, suggesting this position (16,934 bp) as the true E2A TTS. There are two canonical PASs (AAUAAA; 16,964 and 16950 bp) immediately after the CDS any of which can serve as the PAS without affecting the encoded proteins (**Supplementary Table S2**).

The E2B region transcripts also start with the E2-5'UTR but extend downstream to reach the TTS at 2334 bp in the IM region, which is in the immediate context of a canonical PAS (position 2333 bp) where polyadenylation likely occurs. The E2B transcripts, TRXPT_6 and TRXPT_7, are almost identical except for an extra splice junction at the 3'-end of TRXPT_6 (**Figure 7**). TRXPT_7 has the CP for both classical proteins (pTP and Ad-pol) encoded in this region, with the pTP ORF predicted to be spliced from two exons. The predicted splice junction of pTP is corroborated by our data but the full transcript is markedly longer than the predicted ORF, although the encoded product (pTP) remains unchanged. Ad-pol (polymerase) is encoded downstream of pTP with secondary SC (secSC) usage. The CP of TRXPT_6 slightly differs from TRXPT_7 because a new STC resulting from the extra splice site forms a minimal truncation of the Ad-pol encoded from its secSC.

While both TRXPT_6 and TRXPT_7 have the CP for Ad-pol with secSC usage, in all AdVs studied, the two proteins (pTP and Ad-pol) are encoded by separate mRNAs with identical first three 5' exons and TTS, but different splice junctions to the terminal coding exons. Hence, we checked for a longer splice junction between the third and fourth (terminal) exons of TRXPT_7 with our junction validation method (targeted PCR, cloning, and Sanger sequencing). We discovered a unique splice junction (10,981-7062 bp) not present in our RNA-seq data. If initiated from the E2 TSS and terminated at the E2B TTS, this transcript (TRXPT_31) would encode Ad-pol in its 5'-most CDS (**Figure 7**).

Our RNA-seq data also showed a novel short transcript (TRXPT_15) entirely nested within the terminal exon of TRXPT_7 but with a unique splice site. This transcript is an incomplete construction from the mapped reads as it contains a truncated CDS. However, we validated this splice junction to be genuine (**Supplementary PCR methods**).

The IM region is a single-transcript TU, encoding a single classical protein, IVa2. The promoter expressing this single transcript (TRXPT_5) is embedded in the E2B region and shares a TTS with E2B transcripts [@Donovan2020; @Zhao2014]. TRXPT_5 is a two-exon transcript with a non-coding first exon, except the last 2 nucleotides, which connect with the first nucleotide of the second exon to form the 5'-most SC. This new SC is four codons upstream and in-frame of the predicted IVa2 SC. Beside the four additional N-terminus residues, the protein sequence is unchanged.

**Early Region 3 (E3) transcripts**   
The E3 region, nested within the MLTU, encodes proteins that modulate and evade host immune defenses. In MAdVs, this region contains seven ORFs expressed from multiple transcripts sharing the same TSS (from the E3 promoter) but having different TTSs [@Guimet2016; @Donovan2020; @Zhao2014]. However, some E3 transcripts use the TSS of the MLP. Due to sharing the same TSS, in MAdVs, secSC usage is heavily relied on for gene expression in this region as utilizing only the first SC cannot produce the downstream proteins in this TU. The 12.5K ORF and transcripts using the MLP TSS are exceptions [@Donovan2020].

In THEV, only one ORF (E3) was predicted in this region. However, as the E3 TU is nested in the MLTU, transcripts from the L4 promoter (100K, 22K, 33K, and pVIII) overlap the E3 region transcripts entirely and share similar TSS and TTS locations (**Figure 8**). Therefore, we have categorized these two groups together as E3 transcripts.

We identified seven novel transcripts (`r paste0("TRXPT_", c(22:27, 29), collapse = ", ")`) from our RNA-seq data, all originating from two distinct TSSs. We consider the first TSS (position 18,230 bp) as corresponding to the L4 promoter (L4P) and the other at 18,727 bp as corresponding to the E3 promoter (E3P). We also identified the canonical or other known PASs [@Beaudoing2000] near the TTS of the transcripts (see **Supplementary Table S2**). These E3 transcripts collectively have the CP for several predicted THEV ORFs: 100K, 22K, 33K, pVIII, and E3, as well as Fiber (IV) and ORF7 of the MLTU (see **Supplementary Table S3**). However, some of these CDSs differ from the predictions due to either unknown exons or the presence of an in-frame upstream SC. For instance, we discovered that 33K, predicted to be spliced from two exons, is actually a significantly longer four-exon ORF (e33K; 19.8 kDa, 171 residues) encoded on TRXPT_24. Its first two exons were unknown but the last two match the predicted exons and the CDS is in-frame, albeit the first 20 bp of the predicted 33K (including the SC) is spliced out as part of the second intron of TRXPT_24. TRXPT_24 also has the CP for pVIII and E3 if we consider downstream SC usage. However, we found an upstream in-frame SC for the predicted E3; thus, this longer version of E3 (eE3) is likely the genuinely expressed ORF. TRXPT_29, the shortest transcript in this TU, encodes a novel 73 residue protein (8.3KI) across its two exons using the SC of e33K with a unique STC. TRXPT_23, spliced identically as TRXPT_29, also encodes 8.3KI from its first SC. Similarly, TRXPT_22 encodes a 73 residue novel protein (8.3KII) from its first SC that shares over 80% similarity with 8.3KI, but they differ at the C-terminus. Considering downstream SC usage, both TRXPT_22 and TRXPT_23 can encode pVIII and eE3 in that order, but TRXPT_23 being longer, also has the CP for the Fiber ORF.

As the splice junctions of TRXPT_22, TRXPT_23, TRXPT_24, and TRXPT_29 share the same genomic space, their validation was done with a single primer pair, and they were differentiated from each other by cloning the cDNA and Sanger sequencing (**Supplementary PCR methods**). In addition to corroborating the splice junctions for the aforementioned transcripts, the Sanger sequencing results also showed a distinct splice variant undetected in our RNA-seq transcriptome. This was a three-exon transcript (TRXPT_30) with identical first and last exons as TRXPT_23, which also contained the second exon of TRXPT_24 (**Figure 8**). TRXPT_30 encodes a novel 140 residue, 15.7kDa protein (e22K), spanning all three exons. Interestingly, the last 81 C-terminus residues of e22K are identical to 22K (89 residues), a single-exon ORF predicted to use the same SC as 33K. Just as seen for 33K, the first 20 bp of 22K is intronic, excluding the first 7 residues of 22K from e22K. We consider e22K as a long variant of the predicted 22K ORF. Assuming TRXPT_30 shares the same TSS and TTS as TRXPT_23, it would also have the downstream CP of TRXPT_23.

TRXPT_25, the largest transcript in the TU, is a two-exon transcript, encoding a novel protein (t100K; 543 residues), which is a shorter isoform of the predicted 100K ORF. secSC usage on this transcript yields the predicted 22K ORF. It also has the CP for pVIII and eE3 downstream. Furthermore, during the validation of the TRXPT_25 splice junction using primers that span its junction (18,350-18,717 bp), we noticed a DNA band corresponding to the full unspliced sequence (**Supplementary PCR methods**). As TRXPT_25 only falls short of encoding the complete predicted 100K protein due to its splice junction, this band (which we cloned and validated by Sanger sequencing) suggests that the predicted 100K is indeed expressed. We assume that this transcript (TRXPT_25B) shares the same TSS and TTS as TRXPT_25. 

Lastly, TRXPT_26 and TRXPT_27, both originate from the E3P but have distinct TTSs. TRXPT_26 encodes pVIII as the 5'-most ORF and has the CP for eE3 and Fiber in that order. TRXPT_27, a two-exon transcript, encodes Fiber as the 5'-most ORF, and ORF7 downstream with secSC usage. TRXPT_13 is an L4P transcript that uses the MLP TSS; it is discussed under the MLTU transcripts.

**Early Region 4 (E4) transcripts** 
This TU is found at the 3'-end of the genome and expressed on the anti-sense strand. Based on nucleotide position, ORF7 and ORF8 were predicted in this region [@Davison2003]; however, as ORF7 is neither on the anti-sense strand nor transcribed from a promoter in the E4 region, we only classify ORF8 in this TU. This is corroborated by our RNA-seq data, showing only one transcript in this region on the anti-sense strand (**Figure 9**). The transcript (TRXPT_28) spans 25192-26247 bp and is spliced at 25701-26055 bp, forming a two-exon transcript. The second exon fully matches the predicted ORF8 with 12 extra base pairs at the 3'-end. However, we identified a SC 192 bp upstream of the predicted SC in the first exon from which an in-frame protein is encoded. We consider this longer isoform (eORF8 -- 26.4 kDa, 229 residues) as the genuinely expressed ORF. We also identified a canonical PAS 11 bp upstream of TTS (**Supplementary Table S2**).

**MLTU or MLP Region transcripts**   
The MLTU transcripts, dominant in the late phase of the AdV infectious cycle, are produced by alternative polyadenylation and splicing of a primary transcript and grouped into five transcript classes (L1-L5). About 13 out of the 23 predicted ORFs in THEV fall within this TU, some of which we have categorized under the E3 TU instead. Our RNA-seq data revealed 12 transcripts (`r paste0("TRXPT_", c(8:14, 16:20), collapse = ", ")`) in this TU, most of which have the 5' TPL sequence as in all AdVs. However, three transcripts (`r paste0("TRXPT_", c(16:18),collapse = ", ")`) use a different leader sequence (sTPL), where a different first exon is used instead of the first TPL exon, and TRXPT_20 uses only the third TPL exon (TPL3); see **Figure 10**.

We identified five TTSs (10,549, 12,709, 16,870, 22,116, 25,168 bp) in this TU, which we consider as corresponding to the five late mRNA classes (L1-L5), respectively. L1 mRNAs include TRXPT_8, encoding the 52K ORF as predicted. L2 mRNAs include TRXPT_16, TRXPT_17, and TRXPT_18, all containing the sTPL with their respective coding exons. They encode pIIIa, III (penton), and pVII, respectively. The L3 mRNAs, TRXPT_14 and TRXPT_20, both encode the hexon (II) ORF but hexon is the only ORF encoded on TRXPT_14, whereas TRXPT_20 encodes pX (pre-Mu), pVI, and hexon in that order. L4 mRNAs, TRXPT_9, TRXPT_10, TRXPT_11, and TRXPT_13 are the largest transcripts in the transcriptome and encode several similar late proteins. TRXPT_9 and TRXPT_10 are very similar but not identical. The last exon of TRXPT_9 seems to be truncated and likely shares the same TTS as TRXPT_10. They both encode pVII as the 5'-most ORF and also have the CP for pX, pVI, hexon, a longer variant of protease (eProt) from an upstream in-frame SC, and ORF12 (a novel 120 residue protein). Additionally, they have the CP for pVIII and eE3. TRXPT_11 encodes hexon as its 5'-most ORF and also has the CP for eProt, ORF12, e33K, pVIII and eE3. Typically, MLTU transcripts splice the TPL onto a splice site just upstream of the ORF to be expressed [@Donovan2020]. While this holds true for most MLTU ORFs, several late ORFs (pVI, protease, and ORF7) do not have such close proximity splicing but are contained in larger transcripts such as these L4 mRNAs, strongly suggesting the use of non-standard ribosomal initiation mechanisms such as secSC usage or ribosome shunting described in other AdVs for their translation [@Donovan2020; @Yueh1996]. TRXPT_13, an E3 ORF utilizing the MLP TSS, encodes the classical L4P genes, pVIII and eE3. Lastly, the L5 class transcript, TRXPT_12, encodes Fiber as its 5'-most ORF but also has the CP for ORF7. Interestingly, the CP of TRXPT_12 and TRXPT_27 of the E3 TU are identical but are initiated from different TSSs.
\newpage

## DISCUSSION

While the advent of next-generation sequencing has rendered easier the study of large and complex eukaryotic transcriptomes, studying the smaller, compact viral transcriptomes is counterintuitively more challenging, as the transcripts typically have significant overlaps due to genome economization. AdV transcriptomes escalate the difficulty due to the wide array of mRNAs produced via very complex alternative splicing and polyadenylation, all initiated from relatively few promoters. Standard RNA-seq analysis programs, not primarily designed for such compact, gene-dense, and complex transcriptomes, further compound this challenge. Furthermore, in our case, no prior transcriptomic studies for THEV exist; hence, assembling the transcripts without any prior experimentally-derived annotation of THEV splicing using only short illumina reads proved difficult. Lastly, we had initially planned to sequence RNA from another time point (8 hpi); however, all the RNA samples from 8 hpi and one replicate sample from 12 hpi got too degraded during the library preparation steps to be yield any useful data. We believe that these would have contributed to better insights into the temporal expression levels and splicing. Our approach combines standard RNA-seq analysis programs with custom analyses and experimentally validating all splice junctions with independent methods. The transcript map for THEV produced from our analysis is strikingly similar to that of the MAdVs.

Our work provides the first insights into THEV splicing, revealing 34 transcripts grouped into five TUs. The general temporal gene expression regulation observed in MAdVs, with early regions peaking at earlier time points followed by MLTU predominance at later time points, seems to also apply to THEV. An unexpected observation is that the pileup of mapped reads to THEV seems consistently skewed over similar regions of the genome at all time points. Given the temporal regulation of AdVs gene expression, we anticipated distinct differences in read pileups over the genome at different time points, indicating the different stages of infection. This may be due to an unsynchronized infection, leading to transcripts overlapping the time points. Further research is required to determine the precise temporal regulation of THEV.

Short read deep sequencing effectively reconstructs full AdV mRNA structures, particularly mapping splice sites [@Zhao2014]. However, the substantial overlapping nature of AdV mRNAs and fragmentation during library preparation make it challenging to map the exact TSS, TTS, and PASs of assembled transcripts. As AdVs make heavy use of alternative polyadenylation, short read RNA-seq is ill-equipped to discriminate mRNA variants of the same gene produced via alternative polyadenylation. Thus, shorter variants of alternatively polyadenylated mRNAs may potentially be incorporated into the longer variants during transcript assembly, significantly diminishing the diversity of mRNA in the transcriptome. Also, independent transcripts with significant overlaps may be assembled as a single, longer mRNA, since the short reads alone do not provide enough context for the transcript assembler (StringTie) to distinguish them. Such fusions may affect the transcript expression level estimations by inflating or deflating the expression levels of the transcripts involved, affecting the proper understanding of the temporal gene expression regulation and also the diversity of the transcriptome. Transcripts that have reads mistakenly fused with them would have inflated expression levels while those whose reads are counted elsewhere would show false lower expression levels. In our case, since we used other independent methods to validate the splice junctions, we believe these drawbacks to be minimized. Our results show transcripts in the same TU initiating or terminating in similar areas, but not at the exact same position. We consider the most upstream TSS or most downstream TTS for for the transcripts involved but we present them unchanged in all the figures shown (see **Supplementary Table S2**). Also, comparing our results to the better-studied MAdV transcriptomes, we believe some long transcripts in the MLTU (TRXPT_9, TRXPT_10, and TRXPT_11) are likely due to fusing some E3 transcripts to the terminal exons of the MLTU transcripts by StringTie, making them significantly longer. These mRNAs have unusually many exons and their last few exons are identical to some E3 mRNAs. Future studies using long read sequencing technologies will provide more precise mapping of the TSS, TTS, and PASs and clarify the structures of the long MLTU transcripts.

While most predicted ORFs are encoded by the spliced transcripts, we found some that seem to be truncated predictions, as either an upstream in-frame SC or unknown upstream exons were found. Other ORFs were identified that were either shorter or longer isoforms of some predicted ORF. We also found several novel unpredicted ORFs (**Supplementary Table S3**). On this basis, we anticipate that further studies will likely reveal more unpredicted novel ORFs or new variants of predicted ORFs. Furthermore, it is not unreasonable to presume that several splice variants will likely be found as evidenced firstly by finding unique transcripts using 3' RACE and during our splice junction validation steps. And secondly, recent studies [@Donovan2020; @Zhao2014; @Westergren2021] are still discovering novel mRNA variants for even the best studied MAdVs decades later. These new potential proteins and isoforms significantly extend the repertoire of THEV gene products than predicted, adding a hefty number of proteins of undefined function to the previously predicted five (See **Figure 1**). These new potential proteins and isoforms of unknown functions may mediate or contribute to viral replication efficiency, or the immunosuppression associated with THEV. Hence, further studies of these potential proteins is urgently needed.

Eukaryotic mRNAs are typically functionally monocistronic, with the 5’-most AUG determining the translation reading frame. However, AdV mRNAs, which span more than one ORF, are functionally polycistronic, employing non-standard mechanisms of translation initiation such as secSC usage and ribosome shunting [@Westergren2021; @Guimet2016]. AdVs use secondary AUGs as initiation codons for most E1b proteins and some E3 proteins.  In fact, recent studies show that secSC usage is found transcriptome-wide. This is thought to occur because translation initiation at the first SC is inefficient, allowing downstream SCs to be employed [@Donovan2020]. Ribosomal shunting or jumping mechanism is utilized for MLTU transcripts that have the TPL. This mechanism allows the ribosome to translocate to a downstream AUG, under the direction of the shunting elements in the TPL, even if a SC in a good Kozak sequence context is bypassed. Thus, predicting the protein(s) expressed from an AdV mRNA is uncertain as any one of the AUGs may be selected [@Guimet2016; @Westergren2021]. Almost all the THEV transcripts in our data have the CP for several ORFs, some spanning as many as six ORFs. This supports the usage of these special ribosome initiation mechanisms as several predicted and novel ORFs found on mRNA in our data could not be translated using only the typical ribosome scanning mechanism. Interestingly, several distinct THEV mRNAs have identical CPs. This is also observed in human AdVs in a recent study [@Donovan2020]. They proposed that this may permit protein production to be fine-tuned through alteration in the balance between different mRNA groups expressing that ORF.

AdV alternative splicing undergoes a regulated temporal shift in splice site usage, previously thought to be limited to certain TUs. However, recent studies suggest that AdVs routinely produce different combinations of splice acceptor–donor pairs across all TUs [@Westergren2021; @Akusjarvi2008; @Donovan2020; @Guimet2016]. The details of this phenomenon have been best studied for the E1A and L1 units. AdVs modulate the activities of the splicing factor U2AF and the cellular SR family of splicing factors (reviewed here [@Akusjarvi2008]), and encode several proteins that influence the RNA splice site used. This phenomenon appears to occur in the THEV transcriptome, as the stringency of splice acceptor-donor pairs selected decreases from the onset of the late phase (**Figure 5**). Recent studies show that a virtually unlimited number of combinatorial alternative splicing events occur in an AdV lytic infection, resulting in a variety of novel transcripts [@Donovan2020; @Westergren2021]. It is unlikely that the entire repertoire of mRNA produced via this mechanism will actually be translated. However, it has been speculated that the plasticity in alternative RNA splicing enables AdVs to fine-tune protein synthesis by providing different alternatively spliced variants encoding the same protein under changing conditions, conferring an evolutionary advantage [@Donovan2020; @Westergren2021].

## CONCLUSIONS    

The THEV transcriptome bears remarkable similarity to the better-studied MAdVs. The transcriptome is organized into five TUs, with temporal regulation divided into early and late genes, and a broad repertoire of transcripts are produced via virtually unlimited alternative splicing. However, the THEV transcriptome appears less sophisticated (i.e, it encodes fewer genes) than MAdVs, primarily because the MAdV genomes are close to twice as long as that of THEV. The lack of subdivision of the E1 region into E1a and E1b is one of the most obvious examples. Also, the MAdV E4 region encodes several proteins unlike in THEV where only one transcript encoding one protein was found. The complexity of the MLTU leader sequences is another example. While the majority of the THEV MLTU transcripts begin with the TPL just like MAdVs with a small subset using a variant leader sequence (sTPL), significantly more diverse 5'-UTRs are employed for MAdV MLTU transcripts. Namely, the TPL, the so-called x, y, and z leaders, and the i-leader are 5' leaders utilized by MAdV MLTU mRNAs. The absence of these non-TPL leaders in our data could mean that the 5'-UTR diversity of THEV's MLTU mRNAs is more limited due to its smaller genome size or future studies could uncover more variety not seen in our results. We also note that although THEV genomic sequences show minimal differences between strains [@Beach2009a], the transcriptomes may have significant variations; hence, our results may vary from other THEV strains. Also, performing the study *in vivo* or with primary turkey cells may show different results. The potential new proteins identified in our work adds to the number of proteins with undefined functions in THEV; these may have roles in viral replication efficiency, or the immunosuppression associated with THEV. Hence, further studies of these proteins and other predicted proteins of unknown function should be useful in elucidating THEV-induced immunosuppression. Being the first transcriptomic characterization of THEV, this work should serve as useful resource to future THEV gene expression and transcriptomic studies, especially, mapping the mRNA splice sites. More importantly, this work characterizing the splicing of THEV mRNAs will allow researchers to accurately clone any THEV gene(s) of interest to study its potential role in inducing immunosuppression or other functions.

\newpage

## METHODS

### Cell culture and THEV Infection

The Turkey B-cell line (MDTC-RP19, ATCC CRL-8135) was grown as suspension cultures in 1:1 complete Leibovitz's L-15/McCoy's 5A medium with 10% fetal bovine serum (FBS), 20% chicken serum (ChS), 5% tryptose phosphate broth (TPB), and 1% antibiotic solution (100 U/mL Penicillin and 100$\mu g$/mL Streptomycin), at 41^o^C in a humidified atmosphere with 5% CO~2~. Infected cells were maintained in 1:1 serum-reduced Leibovitz's L15/McCoy's 5A media (SRLM) with 2.5% FBS, 5% ChS, 1.2% TPB, and 1% antibiotic solution. A commercially available THEV vaccine was purchased from Hygieia Biological Labs as a source of THEV-A (VAS strain). The stock virus was titrated using an in-house qPCR assay with titer expressed as genome copy number (GCN)/mL, similar to Mahshoub *et al* [@Mahsoub2017] with modifications. Cells were infected in triplicate at MOI of 100 GCN/cell, incubated at 41^o^C for 1 hour, and washed three times with phosphate buffered saline (PBS) to get rid of free virus particles. Triplicate samples were harvested at 4-, 12-, 24-, and 72-hpi for total RNA extraction. The infection was repeated but samples in triplicate were harvested at 12-, 24-, 36-, 48-, and 72-hpi for PCR validation of novel splice sites. Still one more independent infection was done at time points ranging from 12 to 168-hpi for qPCR quantification of virus titers.

### RNA extraction and Sequencing

Total RNA was extracted from infected cells using the Thermofisher RNAqueous™-4PCR Total RNA Isolation Kit (which includes a DNase I digestion step) per manufacturer's instructions. An agarose gel electrophoresis was performed to check RNA integrity. The RNA quantity and purity was initially assessed using nanodrop, and RNA was used only if the A260/A280 ratio was 2.0 ± 0.05 and the A260/A230 ratio was \>2 and \<2.2. Extracted total RNA samples were sent to LC Sciences, Houston TX for poly-A-tailed mRNA sequencing where RNA integrity was checked with Agilent Technologies 2100 Bioanalyzer High Sensitivity DNA Chip and poly(A) RNA-seq library was prepared following Illumina's TruSeq-stranded-mRNA sample preparation protocol. Paired-end sequencing to generate 150 bp reads was performed on the Illumina NovaSeq 6000 sequencing system.

### Validation of Novel Splice Junctions

All splice junctions identified in this work are novel except one predicted splice site each for pTP, DBP, and 33K, which were corroborated in our work. However, these predicted splice junctions had not been experimentally validated hitherto, and we identified additional novel exons, giving the complete picture of these transcripts. The novel splice junctions discovered in this work using the StringTie transcript assembler were validated by PCR, cloning, and Sanger Sequencing (**Supplementary PCR methods**). Briefly, primers spanning a range of novel exon-exon boundaries for each specific transcript in a TU were designed. Universal forward or reverse primers for each respective TU were designed and paired with primers binding specific positions in each transcript. Each forward primer contained a KpnI restriction site and each reverse primer, an XbaI site in the primer 5' ends. After first-strand cDNA synthesis of total RNA obtained from THEV infected MDTC-RP19 cells was done using SuperScript™ IV First-Strand Synthesis System, the primers were used in a targeted PCR amplification, the products analyzed with agarose gel electrophoresis to confirm expected band sizes, cloned by traditional restriction enzyme method, and Sanger sequenced to validate these splice junctions at the sequence level. The total RNA was extracted as described above, including the DNase I digestion step. We included infected total RNA controls with no reverse transcriptase (no RT) during the cDNA synthesis step and the parent RNA were digested using RNase H after cDNA synthesis was complete to ensured that the bands obtained from the targeted PCR amplifications did not originate from the viral genomic DNA. As seen in the agarose gel images in **Supplementary PCR methods**, DNA bands were not found in the "no RT" controls, indicating that the DNA bands seen are of cDNA origin.

### 3' Rapid Amplification of cDNA Ends (3' RACE)

A rapid amplification of sequences from the 3' ends of mRNAs (3' RACE) experiment was performed using a portion of the extracted total RNA of infected MDTC-RP19 cells used for the RNA-seq experiment as explained above. We followed the protocol described by Green *et al* [@Green2019] with modifications. Briefly, 1$\mu g$ of total RNA was reverse transcribed to cDNA using SuperScript™ IV First-Strand Synthesis System following the manufacturing instructions using an adapter-primer with a 3'-end poly(T) and a 5'-end BamHI restriction site. A gene-specific sense primer with a 5'-end KpnI restriction site paired with an anti-sense adapter-primer with a 5'-end BamHI site were used to amplify target sections of the cDNA using Invitrogen's Platinum™ Taq DNA polymerase High Fidelity, following manufacturer's instructions. The PCR amplicons were restriction digested, cloned, and Sanger sequenced.

### Computational Analysis of RNA Sequencing Data: Mapping and Transcript characterization

Sequencing reads were analyzed following a well-established protocol described by Pertea *et al* [@Pertea2016], using Snakemake - version 7.24.0 [@Snakemake2021], a popular workflow management system to drive the pipeline. Briefly, sequencing reads were trimmed with the Trim-galore - version 0.6.6 [@TrimGalore] program to achieve an overall Mean Sequence Quality (Phred Score) of 36. Trimmed reads were mapped simultaneously to the complete genomic sequence of avirulent THEV (<https://www.ncbi.nlm.nih.gov/nuccore/AY849321.1/>) and *Meleagris gallopavo* (<https://www.ncbi.nlm.nih.gov/genome/?term=Meleagris+gallopavo>) using Hisat2 - version 2.2.1 [@Pertea2016] with default settings. The generated binary alignment (BAM) files from each infection time point were filtered for reads mapping to the THEV genome using Samtools - version 1.16.1 and fed into StringTie - version 2.2.1 [@Pertea2016] to assemble the transcripts, using a gene transfer format (GTF) annotation file derived from a gene feature format 3 (GFF3) annotation file obtained from NCBI, which contains the predicted ORFs of THEV as a guide. GFFCOMPARE - version 0.12.6 was used to merge all transcripts from all time points without redundancy and using a custom R script, adenovirus transcripts units (regions) were assigned to each transcript, generating the transcriptome of THEV. StringTie was set to expression estimation mode to calculate FPKM scores for all transcripts after which Ballgown - version 2.33.0 in R was used to perform the statistical analysis on the transcript expression levels. Samtools was also used to count the total sequencing reads for all replicates at each time point and Regtools - version 1.0.0 was used to count all junctions, the reads supporting them, and extract all other information related to the junction. See **Supplementary Computational Analysis** for the details of transcript expression level estimations and splice junction read counts.
\newpage

## LIST OF ABBREVIATIONS  
```{r list_abbs}
set_flextable_defaults(font.size = 10)
tribble(~Abbreviation, ~Definition,
               "AdV", "Adenovirus",
               "MAdV", "Mastadenovirus",
               "SiAdV", "Siadenovirus",
               "THEV", "Turkey Hemorrhagic Enteritis Virus",
              "HE", "Hemorrhagic Enteritis",
              "UTR", "Untranslated Region",
              "TU", "Transcription Unit",
              "L4P", "L4 Promoter",
              "MLP", "Major Late Promter",
              "E3P", "E3 Promoter",
              "hpi", "Hours Post-infection",
              "qPCR", "Quantitative Polymerase Chain Reaction",
              "FPKM", "Fragments Per Kilobase of transcript per Million mapped reads",
              "CP", "Coding Potential",
              "TSS", "Transcription Start Site",
              "TTS", "Transcription Termination Site",
              "SC", "Start Codon", 
              "STC", "Stop Codon",
              "secSC", "Secondary Start Codon",
              "secSTC", "Secondary Stop Codon",
              "ORF", "Open Reading Frame", 
              "CDS", "Coding Sequence", 
              "MLTU", "Major Late Transcription Unit",
              "TPL", "Tripartite Leader",
              "sTPL", "Short Tripartite Leader",
              "TPL3", "Third exon of Tripartite Leader",
              "GCN", "Genome Copy Number") %>%
  flextable::flextable() %>%
  flextable::theme_box() %>%
  flextable::width(width = 1, j = 1) %>%
  flextable::width(width = 2, j = 2) %>%
  flextable::align(align = "center", part = "all")
```

## DATA AVAILABILITY

The raw sequencing read data (FastQ), transcript expression counts, and total unique junctions have been deposited at the National Center for Biotechnology Information Gene Expression Omnibus (<http://www.ncbi.nlm.nih.gov/geo>) under accession number GSE254416.

Data is also available on request by contacting the designated corresponding author.

## CODE AVAILABILITY

All the code/scripts in the entire analysis pipeline are available on github (<https://github.com/Abraham-Quaye/thev_transcriptome>)

## ACKNOWLEDGMENTS

We thank the Office of Research Computing at Brigham Young University for granting us access to the high-performance computing systems to perform the memory-intensive steps in the analysis pipeline of this work.
\newpage

## REFERENCES

```{=tex}
\setlength{\parindent}{-0.25in}
\setlength{\leftskip}{0.25in}
```
\noindent

::: {#refs}
:::

```{=tex}
\setlength{\parindent}{0in}
\setlength{\leftskip}{0in}
\newpage
```
\newpage

## TABLES AND FIGURES

![**Figure 1. Predicted ORF map of THEV avirulent strain.** The central horizontal line represents the double-stranded DNA marked at 2kb intervals as white line breaks. Colored blocks represent viral genes. Blocks above the DNA line are transcribed on the sense DNA strand and those below, on the anti-sense strand. pTP, DBP and 33K are predicted to be spliced and are shown as two exons connected with dashed lines. Shaded regions indicate regions containing the five genus-specific genes of undefined functions (colored red). Genes colored in blue are "genus-common". The gene colored in light green is conserved in all but Atadenoviruses. Regions comprising the different TUs are labelled at the bottom (E1, E2A, E2B, E3, E4, and IM); the unlabeled regions comprise the MLTU.](results/r/figures/figure1.png)

![**Figure 2: Increasing levels of THEV over time. A) Per base coverage of sequence reads mapping to THEV genome by time point.** The pileup of mRNA reads mapping to THEV genome at the base-pair level for each indicated time point. **B) Growth curve of THEV (VAS vaccine strain) in MDTC-RP19 cell line.** Virus quantities in the freeze-thawed supernatant from infected cells were a quantified with a qPCR assay. There is no discernible increase in virus titer up 12 hpi, after which a steady increase in virus titer is measured. The virus titer expands exponentially beginning from 48 hpi, increasing by orders of magnitude before reaching a plateau at 120 hpi. GCN: genome copy number.](results/r/figures/figure2.png)

![**Figure 3. A) Transcriptome of THEV from RNA-seq.** THEV transcripts assembled from all time points by StringTie are unified forming this transcriptome (splicing map). Transcripts belonging to the same TU are located in close proximity on the genome and are color coded and labeled in this figure as such. The organization of TUs in the THEV genome is unsurprisingly similar to MAdVs; however, the MAdV genome shows significantly more transcripts. The TUs are color coded: E1 transcripts - red, E2 - black, E3 - dark grey, E4 - green, MLTU - blue. Predicted ORFs are also indicated here, colored light grey. **B) THEV transcripts identified at given time points.** Transcripts are color coded as explained in **(A)**.](results/r/figures/figure3.png)

![**Figure 4: Changes in splicing and expression profile of THEV over time.** **A) Normalized (FPKM) expression levels of transcripts over time.** The expression levels (FPKM) of individual transcripts as a percentage of the total expression of all transcripts at each time point are indicated. Only transcripts from our RNA-seq data are included here. **B) Normalized (FPKM) expression levels of transcripts by region over time.** The expression levels of each region/TU as a percentage of the total expression of all transcripts at each time point are indicated. Region expression levels were calculated by summing up the FPKMs of all transcripts categorized in that region. **C) Relative abundances of all splice junctions grouped by region/TU over time.** After assigning all `r total_unq_jncs_submit %>% nrow() %>% comma(.)` unique junctions to a TU and the total junction reads counted at each time point for each region, the total junction reads for each TU were plotted as percentages of all junction reads at each time point. Note that the junction read counts are not normalized. **D) Relative abundances of junctions in transcriptome grouped by region/TU over time.** This is identical to **(C)**, except that only the junctions found in the full transcriptome obtained from the RNA-seq data were included.](results/r/figures/figure4.png)

![**Figure 5: Changes in splice donor-acceptor nucleotides over time.** The splice donor-acceptor nucleotides of THEV just like other AdVs is mostly the canonical GU-AG. At early time points (4h.p.i and 12h.p.i **(A)** and **(B)**, respectively) the junction nucleotides used appear to be well scrutinized or restricted, utilizing mostly the canonical splice nucleotides. However, as the infection progresses to the late stages (24h.p.i and 72h.p.i **(C)** and **(D)**, respectively), the selectivity of specific splice acceptor-donor pairs seems to degenerate significantly, such that all combinations of nucleotides are utilized.](results/r/figures/figure5.png)

![**Figure 6: The splice map of the E1 TU.** Exons are depicted as boxes connected by introns (dotted lines). Transcripts from RNA-seq data are colored red, predicted ORFs are colored grey, and transcripts or ORFs discovered by other means are colored black. Each transcript or ORF is labelled with its name to the right. The SC and STC of the 5'-most CDS of each transcript is indicated with the nucleotide position in brackets. The region of the virus is depicted at the bottom as a black line with labels of the nucleotide positions for reference. The table shows sequence reads covering the splice junctions with information about their validation status using cloning and Sanger sequencing.](results/r/figures/figure6.png)

![**Figure 7: The splice map of the E2 and IM TUs.** Exons are depicted as boxes connected by introns (dotted lines). Red transcripts are generated from RNA-seq data and predicted ORFs are colored grey. TRXPT_21B discovered by 3'RACE is colored black. Each transcript or ORF is labelled with its name to the right. The SC and STC of the 5'-most CDS of each transcript are indicated with the nucleotide position in brackets. The region of the virus is depicted at the bottom as a black line with labels of the nucleotide positions for reference. The table shows sequence reads covering the splice junctions with information about their validation status using cloning and Sanger sequencing.](results/r/figures/figure7.png)

![**Figure 8: The splice map of the E3 TU.** Exons are depicted as boxes connected by introns (dotted lines). Red transcripts are generated from RNA-seq data and predicted ORFs are colored grey. Transcripts discovered by other means are colored black. Each transcript or ORF is labelled with its name to the right. The SC and STC of the 5'-most CDS of each transcript are indicated with the nucleotide position in brackets. Similarly, the secondary SC (secSC) and secondary STC (secSTC) are shown. The region of the virus is depicted at the bottom as a black line with labels of the nucleotide positions for reference. The table shows sequence reads covering the splice junctions with information about their validation status using cloning and Sanger sequencing.](results/r/figures/figure8.png)

![**Figure 9: The splice map of the E4 TU.** Exons are depicted as boxes connected by introns (dotted lines). The transcript from RNA-seq data is colored red and the predicted ORF, grey. The transcript and ORF are labelled with their names to the right. The SC and STC of the 5'-most CDS are indicated with the nucleotide position in brackets. The region of the virus is depicted at the bottom as a black line with labels of the nucleotide positions for reference. The table shows sequence reads covering the splice junction with its validation status using cloning and Sanger sequencing.](results/r/figures/figure9.png)

\newpage

![**Figure 10: The splice map of the MLTU.** Exons are depicted as boxes connected by introns (dotted lines). The transcripts from our RNA-seq data are colored red and the predicted ORFs, grey. The transcripts and ORFs are labelled with their names to the right. The SC and STC of the 5'-most CDS of each transcript is indicated with the nucleotide position in brackets. Similarly, the secondary SC (secSC) and secondary STC (secSTC) are shown. The region of the virus is depicted at the bottom as a black line with labels of the nucleotide positions for reference. The table shows sequence reads covering the splice junctions with information about their validation status using cloning and Sanger sequencing.](results/r/figures/figure10.png)

\newpage

\FloatBarrier

```{r table1}
#| tab.id: NULL
#| tab.cap.pre: ""
#| tab.cap.sep: ""

set_flextable_defaults(font.size = 8, table.layout = "fixed", float = "none")
flextable(tab1) %>% theme_zebra() %>%
  flextable::font(part = "all", fontname = "Avenir") %>%
  flextable::width(., j = ~ Metric, width = 1.5) %>%
  flextable::set_caption("Table 1: Overview of sequencing results",
              style = "Table Caption",
              align_with_table = F) %>%
  flextable::bold(part = "body", j = ~ Metric) %>%
  flextable::set_table_properties(layout = "fixed", align = "left",
                       opts_pdf = list(caption_repeat = F),
                       opts_word = list(split = F,
                                        keep_with_next = T))

```

\FloatBarrier

\newpage

```{r}
#| tab.id: NULL
#| tab.cap.pre: ""
#| tab.cap.sep: ""
set_flextable_defaults(font.size = 8)
top_juncs_tp <- function(tab, title){
  tab %>% 
    select(-c(`Splice \n Acceptor-Donor`, Reads, Splice_Site)) %>% 
    dplyr::rename("Reads (Percentage)" = Reads_Percentage) %>%
    flextable() %>%
    theme_zebra() %>%
    flextable::font(part = "all", fontname = "Avenir") %>%
    flextable::set_caption(title,
                style = "Table Caption",
                align_with_table = F) %>%
    flextable::align(align = "center", part = "all") %>%
    flextable::width(., width = 0.7) %>%
    flextable::width(., j = ~ `Reads (Percentage)`, width = 1.2) %>%
    flextable::set_table_properties(layout = "fixed", align = "left",
                       opts_pdf = list(caption_repeat = F),
                       opts_word = list(split = F,
                                        keep_with_next = T))
}
title12 <- "Table 2a: Most abundant splice junctions at 12h.p.i"
top_juncs_tp(sig_12_juncs, title12)
```

\newpage

```{r}
#| tab.id: NULL
set_flextable_defaults(font.size = 8)
title24 <- "Table 2b: Most abundant splice junctions at 24h.p.i"
top_juncs_tp(sig_24_juncs, title24)
```

\newpage

```{r}
#| tab.id: NULL
set_flextable_defaults(font.size = 8)
title72 <- "Table 2c: Most abundant splice junctions at 72h.p.i"
top_juncs_tp(sig_72_juncs, title72)
```

\newpage

## SUPPLEMENTARY MATERIALS

**Supplementary Table S1A**
```{r Sup Table S1A}
set_flextable_defaults(font.size = 8)
make_reg_stats_table <- function(tp_table){
  return(tp_table %>%
    select(-junc_freq) %>% 
    mutate(region = ifelse(is.na(region), "Unassigned", region)) %>% 
    flextable() %>% 
    theme_zebra() %>% 
    set_header_labels(timepoint = "Time", region = "Region", strand = "Strand",
                      sum_reads_reg = "Total Reads", perc_reg = "Percentage") %>% 
    flextable::width(width = 0.8) %>% 
    flextable::align(align = "center", part = "all") %>%
    flextable::set_table_properties(layout = "fixed", align = "left",
                       opts_pdf = list(caption_repeat = FALSE),
                       opts_word = list(split = F))
  )
}
make_reg_stats_table(tp_reg_expr_12) %>%
  set_caption("Table S1a: Most Transcriptionally Active Regions of THEV at 12h.p.i")

```


**Supplementary Table S1B**
```{r Sup Table S1B}
set_flextable_defaults(font.size = 8)
make_reg_stats_table(tp_reg_expr_24) %>%
  set_caption("Table S1b: Most Transcriptionally Active Regions of THEV at 24h.p.i")
```


**Supplementary Table S1C**
```{r Sup Table S1C}
set_flextable_defaults(font.size = 8)
make_reg_stats_table(tp_reg_expr_72) %>%
  set_caption("Table S1c: Most Transcriptionally Active Regions of THEV at 72h.p.i")
```


**Supplementary Table S2**
```{r}
tibble(region = c("E1", "E2", "E2/IM", rep("E3", 3), "E4", rep("MLTU", 5)),
       sub_region = c("E1", "E2A", "E2B/IM", "E3A", "E3B", "E3C", "E4",
                      "L1", "L2", "L3", "L4", "L5"),
       trxpts = c(list(paste0("TRXPT_", 1:4, collapse = ", ")),
                  list(paste0("TRXPT_", c(21, "21B"), collapse = ", ")),
                  list(paste0("TRXPT_", c(5:7, 31), collapse = ", ")),
                  list(paste0("TRXPT_", c(22, 24), collapse = ", ")),
                  list(paste0("TRXPT_", c(23,25,"25B",26,30), collapse = ", ")),
                  "TRXPT_27", "TRXPT_28", "TRXPT_8",
                  list(paste0("TRXPT_", c(16:17), collapse = ", ")),
                  list(paste0("TRXPT_", c(14,20), collapse = ", ")),
                  list(paste0("TRXPT_", c(9:11, 13), collapse = ", ")),
                  "TRXPT_12"),
       tts = c(2325, 16934, 2334, 22491, 23884, 25168, 25192, 10549, 12709,
               16870, 22116, 25168),
       nearest_pas = c("2,323", "16,950", "2,333", "22,458 (AGUAAA)", "23,889",
                       "25,166", "25,203", "10,537", "-", "16,903",
                       "22,098 (AAGAAA)", "25,166")
             ) %>%
  dplyr::mutate(trxpts = as.character(trxpts)) %>%
  flextable(.) %>%
  flextable::theme_zebra() %>%
  flextable::set_header_labels(region = "Region", sub_region = "Sub Region",
                    trxpts = "Transcripts", tts = "TTS",
                    nearest_pas = "Nearest ‘AAUAAA’ TTS signal (location refers to 'U' in the sequence)") %>%
  flextable::width(width = 0.8) %>%
  flextable::width(j = "trxpts", width = 2) %>%
  flextable::width(j = "nearest_pas", width = 1.5) %>%
  flextable::align(align = "center", part = "all") %>%
  flextable::font(fontname = "Arial", part = "all") %>%
  set_caption("Table S2: Mapping Transcript TTS with Closest Polyadenylation Signal")

```


**Supplementary Table S3**
```{r}
set_flextable_defaults(font.size = 10)
#annotate novel proteins with 1 and longer variants with 2 
# fix label clip of fig 7
tibble(trxpts = paste0("TRXPT_", c(1, 2, "2B", 3:21, "21B", 22:25, "25B", 26:31)),
       primary_prod = c("ORF9", "ORF10", "eORF1", "ORF4", "ORF11", "IVa2",
                        rep("pTP", 2), "52K", rep("pVII", 2), "Hexon (II)",
                        "Fiber (IV)", "pVIII", "Hexon (II)", "-", "pIIIa",
                        "Penton (III)", "pVII", "-", "pX", "DBP", "tDBP",
                        "8.3KII", "8.3KI", "e33K", "t100K", "100K", "pVIII",
                        "Fiber (IV)", "eORF8", "8.3KI", "e22K", "Ad-pol"
                        ),
       size_kda = c(17.9, 66.4, 64.3, 13.1, 15.9, 42.3, rep(70.5, 2), 33.8,
                    rep(13.2, 2), 101.1, 49.0, 21.8, 101.1, "-", 57.6, 50.8,
                    13.2, "-", 6.1, 43.3, 39.3, 8.6, 8.3, 19.8, 62.4, 76.5,
                    21.8, 49.0, 26.4, 8.3, 15.7, 129.2
                    ),
       size_aa = c(160, 597, 580, 115, 143, 371, rep(597, 2), 300, rep(120, 2),
                   906, 454, 200, 906, "-", 505, 448, 120, "-", 58, 380, 346,
                   73, 73, 171, 543, 665, 200, 454, 229, 73, 140, 1112
                   ),
       start = c(211, 211, 1965, 211, 211, 3616, rep(10995, 2), 8570,
                 rep(12347, 2), 13610, 22519, 20769, 13610, "-", 9462, 11001,
                 12347, "-", 12712, 18186, 18013, rep(18230, 5), 20769, 22519,
                 26246, rep(18230, 2), 6768
                 ),
       stop = c(rep(2312, 2), 1953, 2312, 1953, 2334, rep(6765, 2), 9472,
                rep(12709, 2), 16330, 23883, 21371, 16330, "-", 10979, 12347,
                12709, "-", 12888, rep(16973, 2), 20457, 20262, 20699, rep(20227, 2),
                21371, 23883, 25204, 20262, 20411, 3430
                ),
       coding_pt = c(rep(list(c("Hyd", "ORF4")), 3), "Hyd", list(c("Hyd", "ORF4")),
                     "-", rep("Ad-pol", 2), "-",
                     rep(list(c("pX", "pVI", "hexon", "eProt", "ORF12", "pVIII", "eE3")), 2),
                     list(c("eProt", "ORF12", "e33K", "pVIII", "eE3")), "ORF7",
                     "eE3", "-", "-", list(c("Penton (III)", "pVII")), "-", "-",
                     "-", list(c("pVI", "hexon")), "-", "-", list(c("pVIII", "eE3")),
                     list(c("pVIII", "eE3", "Fiber")), list(c("pVIII", "E3")),
                     rep(list(c("22K", "pVIII", "eE3")), 2), list(c("eE3", "Fiber")),
                     "ORF7", "-", "-", list(c("pVIII", "eE3", "Fiber")), "-"
                     )
       ) %>%
  mutate(coding_pt = as.character(coding_pt)) %>%
  mutate(coding_pt = str_replace(coding_pt, '^c\\((.*)\\)$', '\\1')) %>%
  mutate(coding_pt = str_replace_all(coding_pt, '"', "")) %>% 
  flextable(.) %>%
  flextable::theme_zebra() %>%
  flextable::set_header_labels(trxpts = "mRNA", primary_prod = "Primary Product",
                               size_kda = "Size\n(kDa)", 
                               size_aa = "Number of\nResidues",
                               start = "Start Codon\nPosition",
                               stop = "Stop Codon\nPosition",
                               coding_pt = "Downstream\nCoding Potential") %>% 
  flextable::width(width = 0.8) %>%
  flextable::width(j = "coding_pt", width = 2) %>%
  flextable::align(align = "center", part = "all") %>%
  flextable::font(fontname = "Arial", part = "all") %>%
  footnote(., i = c(1,2,5,24,25), j = 2,
           value = as_paragraph(c("Novel product identified in this work")),
           ref_symbols = c("new"), part = "body", inline = T) %>%
  footnote(., i = c(3,26,31,33), j = 2,
           value = as_paragraph(c("Longer isoform identified in this work")),
           ref_symbols = c("L"), part = "body", inline = F) %>%
  footnote(., i = c(23), j = 2,
           value = as_paragraph(c("Shorter isoform identified in this work")),
           ref_symbols = c("S"), part = "body", inline = F) %>%
  set_caption("Table S3: The Encoded Proteins and Coding Potential of THEV Transcripts")
```

\newpage

**Supplementary PCR Methods**

```{r Supplementary PCR Methods}
set_flextable_defaults(font.size = 3.5)

supp_pcr_meth_tab %>% 
  select(-valid_status) %>% 
flextable() %>%
  theme_zebra() %>%
  set_header_labels(trxpt_id = "Transcript ID", region = "Region", num_exons = "Number of Exons",
                    full_trxpt = "Full Transcript", exon1 = "Exon 1", exon2 = "Exon 2",
                    exon3 = "Exon 3", exon4 = "Exon 4", exon5 = "Exon 5", exon6 = "Exon 6",
                    exon7 = "Exon 7", forwardP = "Forward Primer", reverseP = "Reverse Primer",
                    gel_image = "Agarose Gel") %>% 
  flextable::fontsize(j = c("full_trxpt", "exon1", "exon2", "exon3",
                            "exon4", "exon5", "exon6", "exon7",
                            "forwardP", "reverseP"), size = 3) %>%
  flextable::align(align = "center", part = "all") %>%
  flextable::width(., width = 0.35) %>% 
  colformat_image(j = "gel_image", i = c(1:3, 5:26, 28:30),
                  width = 1.1, height = 1) %>%
  flextable::width(j = ~forwardP, width = 0.6) %>%
  flextable::width(j = ~reverseP, width = 0.6) %>%
  flextable::width(j = ~gel_image, width = 1.15) %>%
  # flextable::height(height = 0, part = "header") %>% 
  footnote(., i = c(1:26, 28:30), j = 12, value = as_paragraph(c("Primer binds inside first exon")),
           ref_symbols = c("a"), part = "body", inline = T) %>%
  footnote(., i = c(1:7, 9:18, 21:26, 28, 29), j = 13, value = as_paragraph(c("Primer binds inside terminal exon")), ref_symbols = c("b"), part = "body", inline  = T) %>% 
  # transcripts too long to validate with one primer pair
  footnote(., i = c(8,19, 20, 30), j = 13, value = as_paragraph(c("Primer binds inside fourth exon")),
           ref_symbols = c("c"), part = "body", inline = T) %>% 
  footnote(., i = c(8), j = 14, value = as_paragraph(c("Agarose gel identical to TRXPT_7 due to identical splicing")), ref_symbols = c("I"), part = "body", inline = T) %>%
  footnote(., i = c(10), j = 14, value = as_paragraph(c("Agarose gel identical to last 3 exons of TRXPT_10 due to identical splicing")), ref_symbols = c("II"), part = "body", inline = T) %>%
  footnote(., i = c(12), j = 14, value = as_paragraph(c("Agarose gel identical to last 4 exons of TRXPT_11 due to identical splicing")), ref_symbols = c("III"), part = "body", inline = T) %>% 
  footnote(., i = c(16), j = 14, value = as_paragraph(c("Agarose gel dentical to TRXPT_23 due to identical splicing")), ref_symbols = c("IV"), part = "body", inline = T) %>% 
  footnote(., i = c(19), j = 14, value = as_paragraph(c("Agarose gel identical to TRXPT_9 due to identical splicing")), ref_symbols = c("V"), part = "body", inline = T) %>% 
  footnote(., i = c(20), j = 14, value = as_paragraph(c("Agarose gel identical to TRXPT_14 due to identical splicing")), ref_symbols = c("VI"), part = "body", inline = T) %>%
  flextable::set_table_properties(layout = "fixed", align = "left",
                       opts_pdf = list(caption_repeat = F),
                       opts_word = list(split = F)) %>%
  set_caption("Agarose Gels Showing PCR Amplification of THEV cDNA With Gene-Specific Primers")
```

In the table above, the restriction sites in the primer tails are shown in lowercase letters.
All the primer melting temperatures (TMs) are 58-60^o^C using a hot start Taq DNA polymerase.
The PCR reaction mix was made per manufacturer's instructions. The PCR cycling conditions were as follows: Initial denaturation -- 95^o^C for 1 minute; cyclical denaturation -- 95^o^C for 30 seconds, annealing -- variable temperature (53^o^C-56^o^C) for 30 seconds, primer extension -- 68^o^C for variable time, and final elongation -- 68^o^C for 5 minutes. We used 35 cycles of amplification. The PCR products of THEV cDNA shown in the gels in the table above were cloned and Sanger sequenced to confirm the splice junctions. We included "No reverse transcriptase (No RT)" template controls to ensure the PCR products are of RNA origin.

**Supplementary Computational Analysis**\
Snakemake v7.24.0 was used to manage our entire workflow. A graph of the main steps in our pipeline generated with Snakemake is shown below. Our trimmed RNA-seq reads were mapped to the genome of *M. gallopavo* (with the THEV genome as one of its chromosomes) using Hisat2, to generate the alignment (BAM) files and StringTie used to assemble the transcriptome with a GTF annotation file containing the predicted THEV ORFs as a guide. The GTF annotation file was derived from a GFF3 annotation file obtained from NCBI using Agat - version 1.0.0, a program for converting between many different file formats used in bioinformatics. However, the NCBI GFF3 annotation file itself was first modified to remove all unimportant features, leaving only the ORFs.

StringTie was also used to estimate the normalized expression levels (FPKM) of all the transcripts and Ballgown in R was used to perform statistical analysis and comparisons of the transcript expression levels, which was instructive in understanding the temporal regulation THEV gene expression.

In these steps above, each sample (replicate of each time point) was processed independently and merged only in the final transcriptome assembly or during analysis with Ballgown. In the subsequent steps described below, all samples for each time point were processed together.

We used RegTools to extract and analyze the splice junctions in the BAM files. The command "regtools junctions extract" provides a wealth of information about all the splice sites in the BAM file provided such as: the start and end positions, the strand, and number of reads supporting the splice junctions. The command "regtools junctions annotate" gives even more information such as: the splice site donor-acceptor sequences and transcripts/genes that overlap the junction. This information was the basis for estimating and comparing the splicing activity of different regions (TUs) of THEV over time. Also, Samtools was also used to count the total sequencing reads for all replicates at each time point.

![A flowchart of the major steps in the computational analysis pipeline (*generated with Snakemake*)](project_map.png)
