---
title: Turkey Hemorrhagic Enteritis Virus transcriptome profiling
author: Abraham Quaye
date: 2023-03-16
output:
    github_document:
        html_preview: FALSE
---
```{css echo=FALSE}
body {
    <!-- background-color: #262626; -->
    <!-- color: white; -->
    font-size: 2vw
}
```
<!-- # THEV transcriptome studies -->

## Background

**Hemorrhagic enteritis (HE)** is a disease affecting 6-12-week-old turkeys characterized by bloody diarrhea, **immunosuppression (IS)**, and up to 60% mortality. This disease is caused by **_Turkey Hemorrhagic Enteritis Virus (THEV)_** of which avirulent strains (THEV-A) that do not cause HE but retain the immunosuppressive ability have been isolated. The THEV-A Virginia Avirulent Strain (VAS) is still used as a live vaccine despite its immunosuppressive properties. **_Our objective is to understand the genetic basis by which VAS brings about IS_**. However, the **transcriptome of THEV** has not been characterized, making the investigation of specific viral genes for possible roles in causing IS impractical as the viral splicing and gene expression patterns are unknown. To identify such viral factors, a well-characterized transcriptome of THEV is required to set the stage for experimentation with specific viral genes that may mediate IS.

## Methods

After infecting a turkey B-cell line (RP-19) with THEV, samples in triplicates were collected at 4-, 12-, 24-, and 72-hours post-infection. Total RNA was subsequently extracted using Thermofishers’s RNAqueous-4PCR Total RNA Isolation Kit, and poly-A-tailed mRNA sequencing done at LC Sciences, Houston TX. The raw sequencing reads were trimmed with the FastQC program, mapped to the THEV genome using Hisat2 and transcripts assembled with StringTie. The `--merge` function of StringTie was used to consolidate transcripts from all timepoints, generating the final THEV transcriptome.

## Data analysis
This project is run from the `Snakefile`. Full analysis pipeline shown below in diagram:

![](./project_map.pdf){width=920px height=680px}

## Results

Over 18 million reads mapped to THEV genome providing good coverage/depth, leaving no regions unmapped. All predicted genes in the genome were represented. Interestingly, all were spliced with either with 5’- or 3’-multi exon UTRs hitherto unknown. Thirteen novel exons were identified which require additional verification. The splicing patterns strongly suggest that there are 3 main promoters (E1a-, E3, and major late promoters) driving expression of most of the genes with 2 possible minor promoters driving single genes (ORF7 and ORF8).

We have performed the first RNA-sequencing experiment aimed at characterizing THEV’s transcriptome, which has yielded the only experimentally determined and the most detailed insight of THEV gene expression. In keeping with all Adenoviruses, our data shows that almost all THEV genes are spliced, and under the control of a promoter, several genes are expressed as one transcription unit. This insight of THEV’s transcriptome may allow the engineering of VAS to provide immune protection with less or no associated IS.

### Navigate the project:

```{r list_dirs, exclude=TRUE, message=FALSE, warning=FALSE, echo=FALSE}
library(tidyverse)
library(rmarkdown)
dirs <- list.dirs(".", recursive = FALSE, full.names = FALSE)
all_files <- list.files(".", all.files = TRUE)
root_files <- subset(all_files, all_files %in% dirs == FALSE)
script_dir <- list.files("scripts")
rawfiles_dir <- list.files("raw_files")
results_dir <- list.files("results")
reads_dir <- list.files("trimmedReads")
```
* **`ROOT (thev_transcriptome)/`**
    * **Directories:**
        * *``r dirs[4]`/`* - Add README for more details
            * ``r rawfiles_dir[1]`/`
            * ``r rawfiles_dir[2]`/`
            * ``r rawfiles_dir[3]`/`
        * *``r dirs[5]`/`* - Add README for more details
            * ``r results_dir[1]`/`
            * ``r results_dir[2]`/`
            * ``r results_dir[3]`/`
        * *``r dirs[6]`/`* - Add README for more details
            * ``r script_dir[1]`/`
            * ``r script_dir[2]`/`
            * ``r script_dir[3]`/`
        * *``r dirs[7]`/`* - Add README for more details
            * ``r reads_dir[1]`/`
            * ``r reads_dir[2]`/`
    * **Files**
        * *``r root_files[5]``* - *`deprecated`*: replaced by `Snakefile`
        * *``r root_files[6]``*
        * *``r root_files[7]``*
        * *``r root_files[8]``*
        * *``r root_files[9]``*
        * *``r root_files[10]``*
        * *``r root_files[11]``*
        * *``r root_files[12]``*
        * *``r root_files[13]``* - *not on Git*
    * **Hidden files** - *not on Git*
        * *``r root_files[3]``*
        * *``r root_files[4]``*
        * ``r dirs[1]`/`
        * ``r dirs[2]`/`
        * ``r dirs[3]`/`

### Dependencies:

- `AGAT`
- `Hisat2`
- `StringTie`
- ``r R.version.string``
- `tidyverse`(v.`r packageVersion("tidyverse")`)
- `rmarkdown`(v.`r packageVersion("rmarkdown")`)
- `Snakemake`
- `zsh shell`
- See `environment.yml` for `conda env` dependencies
