---
title: "Supplementary Materials: Elucidating the Transcriptome of Turkey Hemorrhagic Enteritis Virus"
output: 
  officedown::rdocx_document:
    reference_docx: style_word_output.docx
  pdf_document: default
geometry: margin=1in
font-size: 11pt
header-includes:
  - \usepackage{helvet}
  - \renewcommand*\familydefault{\sfdefault}
  - \usepackage{setspace}
  - \doublespacing
  - \usepackage[left]{lineno}
  - \linenumbers
editor_options:
  chunck_output_type: console
---
\vspace{5mm}

**Running Title:** Novel Insights into Turkey Hemorrhagic Enteritis Virus Transcriptome

\vspace{10mm}
Abraham Quaye^1\*^, Bret Pickett^\*^, Joel S. Griffitts^\*^, Bradford K. Berges^\*^, Brian D. Poole${^\dagger}$^\*^   

\vspace{5mm}

^\*^Department of Microbiology and Molecular Biology, Brigham Young University       
^1^First-author      
${^\dagger}$ Corresponding Author   

\vspace{5mm}
**Corresponding Author Information**     
[brian_poole@byu.edu](mailto:brian_poole@byu.edu)       
Department of Microbiology and Molecular Biology,       
4007 Life Sciences Building (LSB),    
Brigham Young University,   
Provo, Utah  
\newpage

```{r setup, include=FALSE, echo=FALSE, message=FALSE, warning=FALSE}
knitr::opts_chunk$set(echo=FALSE, message=FALSE, warning=FALSE)
library(knitr)
library(tidyverse)
library(flextable)
source("scripts/r/bam_file_analysis.R")
source("scripts/r/reg_by_reg_plots.R")
```


## Supplementary Table S1A

```{r Sup Table S1A}
set_flextable_defaults(font.size = 8)
make_reg_stats_table <- function(tp_table){
  return(tp_table %>%
    select(-junc_freq) %>% 
    mutate(region = ifelse(is.na(region), "Unassigned", region)) %>% 
    flextable() %>% 
    theme_zebra() %>% 
    set_header_labels(timepoint = "Time", region = "Region", strand = "Strand",
                      sum_reads_reg = "Total Reads", perc_reg = "Percentage") %>% 
    flextable::width(width = 0.8) %>% 
    flextable::align(align = "center", part = "all") %>%
    flextable::set_table_properties(layout = "fixed", align = "left",
                       opts_pdf = list(caption_repeat = FALSE),
                       opts_word = list(split = F))
  )
}
make_reg_stats_table(tp_reg_expr_12) %>%
  set_caption("Table S1a: Most Transcriptionally Active Regions of THEV at 12h.p.i")

```

## Supplementary Table S1B

```{r Sup Table S1B}
set_flextable_defaults(font.size = 8)
make_reg_stats_table(tp_reg_expr_24) %>%
  set_caption("Table S1b: Most Transcriptionally Active Regions of THEV at 24h.p.i")
```

## Supplementary Table S1C

```{r Sup Table S1C}
set_flextable_defaults(font.size = 8)
make_reg_stats_table(tp_reg_expr_72) %>%
  set_caption("Table S1c: Most Transcriptionally Active Regions of THEV at 72h.p.i")
```

\newpage
## Supplementary PCR Methods
```{r Supplementary PCR Methods}
set_flextable_defaults(font.size = 3.5)

supp_pcr_meth_tab %>% 
  select(-valid_status) %>% 
flextable() %>%
  theme_zebra() %>%
  set_header_labels(trxpt_id = "Transcript ID", region = "Region", num_exons = "Number of Exons",
                    full_trxpt = "Full Transcript", exon1 = "Exon 1", exon2 = "Exon 2",
                    exon3 = "Exon 3", exon4 = "Exon 4", exon5 = "Exon 5", exon6 = "Exon 6",
                    exon7 = "Exon 7", forwardP = "Forward Primer", reverseP = "Reverse Primer",
                    gel_image = "Agarose Gel") %>% 
  flextable::fontsize(j = c("full_trxpt", "exon1", "exon2", "exon3",
                            "exon4", "exon5", "exon6", "exon7",
                            "forwardP", "reverseP"), size = 3) %>%
  flextable::align(align = "center", part = "all") %>%
  flextable::width(., width = 0.35) %>% 
  colformat_image(j = "gel_image", i = c(1:3, 5:26, 28:30),
                  width = 1.1, height = 1) %>%
  flextable::width(j = ~forwardP, width = 0.6) %>%
  flextable::width(j = ~reverseP, width = 0.6) %>%
  flextable::width(j = ~gel_image, width = 1.15) %>%
  # flextable::height(height = 0, part = "header") %>% 
  footnote(., i = c(1:26, 28:30), j = 12, value = as_paragraph(c("Primer binds inside first exon")),
           ref_symbols = c("a"), part = "body", inline = T) %>%
  footnote(., i = c(1:7, 9:18, 21:26, 28, 29), j = 13, value = as_paragraph(c("Primer binds inside terminal exon")), ref_symbols = c("b"), part = "body", inline  = T) %>% 
  # transcripts too long to validate with one primer pair
  footnote(., i = c(8,19, 20, 30), j = 13, value = as_paragraph(c("Primer binds inside fourth exon")),
           ref_symbols = c("c"), part = "body", inline = T) %>% 
  footnote(., i = c(8), j = 14, value = as_paragraph(c("Agarose gel identical to TRXPT_7 due to identical splicing")), ref_symbols = c("I"), part = "body", inline = T) %>%
  footnote(., i = c(10), j = 14, value = as_paragraph(c("Agarose gel identical to last 3 exons of TRXPT_10 due to identical splicing")), ref_symbols = c("II"), part = "body", inline = T) %>%
  footnote(., i = c(12), j = 14, value = as_paragraph(c("Agarose gel identical to last 4 exons of TRXPT_11 due to identical splicing")), ref_symbols = c("III"), part = "body", inline = T) %>% 
  footnote(., i = c(16), j = 14, value = as_paragraph(c("Agarose gel dentical to TRXPT_23 due to identical splicing")), ref_symbols = c("IV"), part = "body", inline = T) %>% 
  footnote(., i = c(19), j = 14, value = as_paragraph(c("Agarose gel identical to TRXPT_9 due to identical splicing")), ref_symbols = c("V"), part = "body", inline = T) %>% 
  footnote(., i = c(20), j = 14, value = as_paragraph(c("Agarose gel identical to TRXPT_14 due to identical splicing")), ref_symbols = c("VI"), part = "body", inline = T) %>%
  flextable::set_table_properties(layout = "fixed", align = "left",
                       opts_pdf = list(caption_repeat = F),
                       opts_word = list(split = F)) %>%
  set_caption("Agarose Gels Showing PCR Amplification of THEV cDNA With Gene-Specific Primers")
```

In the table above, the restriction sites in the primer tails are shown in lowercase letters. All the primer melting temperatures (TMs) are 58-60^o^C using a hot start Taq DNA polymerase. The PCR reaction mix was done per manufacturer's instructions. The PCR cycling conditions were as follows: Initial denaturation -- 95^o^C for 1 minute; cyclical denaturation -- 95^o^C for 30 seconds, annealing -- variable temperature (53^o^C-56^o^C) for 30 seconds, primer extension -- 68^o^C for variable time, and final elongation -- 68^o^C for 5 minutes. We used 35 cycles of amplification.

## Supplementary Computational Analysis   
`Snakemake v7.24.0` was used to manage our entire workflow. A graph of the main steps in our pipeline generated with `Snakemake` is shown below. Our trimmed RNA-seq reads were mapped to the genome of _M. gallopavo_ (with THEV's genome as one of its chromosomes) using `Hisat2`, to generate the alignment (BAM) files and `StringTie` used to assemble the transcriptome with a `GTF` annotation file containing the predicted THEV ORFs as a guide. The `GTF` annotation file was derived from a `GFF3` annotation file obtained from NCBI using `Agat - version 1.0.0`, a program for converting between many different file formats used in bioinformatics. However, the NCBI `GFF3` annotation file itself was first modified to remove all unimportant features, leaving only the ORFs. 

`StringTie` was also used to estimate the normalized expression levels (FPKM) of all the transcripts and `Ballgown` in `R` was used to perform statistical analysis and comparisons of the transcript expression levels, which instructive in understanding the temporal regulation THEV gene expression. 

In these steps above, each sample (replicate of each time point) was processed independently and merged only in the final transcriptome assembly or during analysis with `Ballgown`. In the subsequent steps described below, all samples for each time point were processed together.

We used `RegTools` to extract and analyze the splice junctions in the BAM files. The command `regtools junctions extract` provides a wealth of information about all the splice sites in the BAM file provided such as: the start and end positions, the strand, and number of reads supporting the splice junctions. The command `regtools junctions annotate` gives even more information such as: the splice site donor-acceptor sequences and transcripts/genes that overlap the junction. These information was the basis for estimating and comparing the splicing activity of different regions (TUs) of THEV over time. Also, `Samtools` was also used to count the total sequencing reads for all replicates at each time point.
![](project_map.png)